From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: liviyifan <344781483+liviyifan@users.noreply.github.com>
Date: Thu, 22 Oct 2026 21:13:58 +0800
Subject: [PATCH 6/7] aarch64: add the ILP32 ABI lists and test data

The abilists start at GLIBC_2.41. libanl, libdl, libnsl, libpthread,
librt and libutil have empty lists, which also keep the LP64 ones in
the parent directory from being used. localplt.data is the LP64 one
without matherr; c++-types.data was generated with the ILP32 g++.
---
 .../sysv/linux/aarch64/ilp32/c++-types.data   |   67 +
 .../unix/sysv/linux/aarch64/ilp32/ld.abilist  |    8 +
 .../aarch64/ilp32/libBrokenLocale.abilist     |    1 +
 .../sysv/linux/aarch64/ilp32/libanl.abilist   |    0
 .../sysv/linux/aarch64/ilp32/libc.abilist     | 2272 +++++++++++++++++
 .../aarch64/ilp32/libc_malloc_debug.abilist   |   26 +
 .../sysv/linux/aarch64/ilp32/libdl.abilist    |    0
 .../sysv/linux/aarch64/ilp32/libm.abilist     | 1126 ++++++++
 .../sysv/linux/aarch64/ilp32/libnsl.abilist   |    0
 .../linux/aarch64/ilp32/libpthread.abilist    |    0
 .../linux/aarch64/ilp32/libresolv.abilist     |   55 +
 .../sysv/linux/aarch64/ilp32/librt.abilist    |    0
 .../linux/aarch64/ilp32/libthread_db.abilist  |   40 +
 .../sysv/linux/aarch64/ilp32/libutil.abilist  |    0
 .../sysv/linux/aarch64/ilp32/localplt.data    |   10 +
 15 files changed, 3605 insertions(+)
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/c++-types.data
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libc_malloc_debug.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/localplt.data

diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/c++-types.data b/sysdeps/unix/sysv/linux/aarch64/ilp32/c++-types.data
new file mode 100644
index 0000000000..d4ab1d164a
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/c++-types.data
@@ -0,0 +1,67 @@
+blkcnt64_t:x
+blkcnt_t:x
+blksize_t:i
+caddr_t:Pc
+clockid_t:i
+clock_t:l
+daddr_t:i
+dev_t:y
+fd_mask:l
+fsblkcnt64_t:y
+fsblkcnt_t:y
+fsfilcnt64_t:y
+fsfilcnt_t:y
+fsid_t:8__fsid_t
+gid_t:j
+id_t:j
+ino64_t:y
+ino_t:y
+int16_t:s
+int32_t:i
+int64_t:x
+int8_t:a
+intptr_t:i
+key_t:i
+loff_t:x
+mode_t:j
+nlink_t:j
+off64_t:x
+off_t:x
+pid_t:i
+pthread_attr_t:14pthread_attr_t
+pthread_barrier_t:17pthread_barrier_t
+pthread_barrierattr_t:21pthread_barrierattr_t
+pthread_cond_t:14pthread_cond_t
+pthread_condattr_t:18pthread_condattr_t
+pthread_key_t:j
+pthread_mutex_t:15pthread_mutex_t
+pthread_mutexattr_t:19pthread_mutexattr_t
+pthread_once_t:i
+pthread_rwlock_t:16pthread_rwlock_t
+pthread_rwlockattr_t:20pthread_rwlockattr_t
+pthread_spinlock_t:i
+pthread_t:m
+quad_t:x
+register_t:x
+rlim64_t:y
+rlim_t:y
+sigset_t:10__sigset_t
+size_t:m
+socklen_t:j
+ssize_t:l
+suseconds_t:x
+time_t:x
+u_char:h
+uid_t:j
+uint:j
+u_int:j
+u_int16_t:t
+u_int32_t:j
+u_int64_t:y
+u_int8_t:h
+ulong:m
+u_long:m
+u_quad_t:y
+useconds_t:j
+ushort:t
+u_short:t
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist
new file mode 100644
index 0000000000..538b50fe49
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist
@@ -0,0 +1,8 @@
+GLIBC_2.41 __libc_stack_end D 0x4
+GLIBC_2.41 __rseq_flags D 0x4
+GLIBC_2.41 __rseq_offset D 0x4
+GLIBC_2.41 __rseq_size D 0x4
+GLIBC_2.41 __stack_chk_guard D 0x4
+GLIBC_2.41 __tls_get_addr F
+GLIBC_2.41 _dl_mcount F
+GLIBC_2.41 _r_debug D 0x14
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist
new file mode 100644
index 0000000000..d988d1d0be
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist
@@ -0,0 +1 @@
+GLIBC_2.41 __ctype_get_mb_cur_max F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist
new file mode 100644
index 0000000000..e69de29bb2
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist
new file mode 100644
index 0000000000..835c853d41
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist
@@ -0,0 +1,2272 @@
+GLIBC_2.41 _Exit F
+GLIBC_2.41 _Fork F
+GLIBC_2.41 _IO_2_1_stderr_ D 0xa0
+GLIBC_2.41 _IO_2_1_stdin_ D 0xa0
+GLIBC_2.41 _IO_2_1_stdout_ D 0xa0
+GLIBC_2.41 _IO_adjust_column F
+GLIBC_2.41 _IO_adjust_wcolumn F
+GLIBC_2.41 _IO_default_doallocate F
+GLIBC_2.41 _IO_default_finish F
+GLIBC_2.41 _IO_default_pbackfail F
+GLIBC_2.41 _IO_default_uflow F
+GLIBC_2.41 _IO_default_xsgetn F
+GLIBC_2.41 _IO_default_xsputn F
+GLIBC_2.41 _IO_do_write F
+GLIBC_2.41 _IO_doallocbuf F
+GLIBC_2.41 _IO_fclose F
+GLIBC_2.41 _IO_fdopen F
+GLIBC_2.41 _IO_feof F
+GLIBC_2.41 _IO_ferror F
+GLIBC_2.41 _IO_fflush F
+GLIBC_2.41 _IO_fgetpos F
+GLIBC_2.41 _IO_fgetpos64 F
+GLIBC_2.41 _IO_fgets F
+GLIBC_2.41 _IO_file_attach F
+GLIBC_2.41 _IO_file_close F
+GLIBC_2.41 _IO_file_close_it F
+GLIBC_2.41 _IO_file_doallocate F
+GLIBC_2.41 _IO_file_finish F
+GLIBC_2.41 _IO_file_fopen F
+GLIBC_2.41 _IO_file_init F
+GLIBC_2.41 _IO_file_jumps D 0x54
+GLIBC_2.41 _IO_file_open F
+GLIBC_2.41 _IO_file_overflow F
+GLIBC_2.41 _IO_file_read F
+GLIBC_2.41 _IO_file_seek F
+GLIBC_2.41 _IO_file_seekoff F
+GLIBC_2.41 _IO_file_setbuf F
+GLIBC_2.41 _IO_file_stat F
+GLIBC_2.41 _IO_file_sync F
+GLIBC_2.41 _IO_file_underflow F
+GLIBC_2.41 _IO_file_write F
+GLIBC_2.41 _IO_file_xsputn F
+GLIBC_2.41 _IO_flockfile F
+GLIBC_2.41 _IO_flush_all F
+GLIBC_2.41 _IO_flush_all_linebuffered F
+GLIBC_2.41 _IO_fopen F
+GLIBC_2.41 _IO_fprintf F
+GLIBC_2.41 _IO_fputs F
+GLIBC_2.41 _IO_fread F
+GLIBC_2.41 _IO_free_backup_area F
+GLIBC_2.41 _IO_free_wbackup_area F
+GLIBC_2.41 _IO_fsetpos F
+GLIBC_2.41 _IO_fsetpos64 F
+GLIBC_2.41 _IO_ftell F
+GLIBC_2.41 _IO_ftrylockfile F
+GLIBC_2.41 _IO_funlockfile F
+GLIBC_2.41 _IO_fwrite F
+GLIBC_2.41 _IO_getc F
+GLIBC_2.41 _IO_getline F
+GLIBC_2.41 _IO_getline_info F
+GLIBC_2.41 _IO_gets F
+GLIBC_2.41 _IO_init F
+GLIBC_2.41 _IO_init_marker F
+GLIBC_2.41 _IO_init_wmarker F
+GLIBC_2.41 _IO_iter_begin F
+GLIBC_2.41 _IO_iter_end F
+GLIBC_2.41 _IO_iter_file F
+GLIBC_2.41 _IO_iter_next F
+GLIBC_2.41 _IO_least_wmarker F
+GLIBC_2.41 _IO_link_in F
+GLIBC_2.41 _IO_list_all D 0x4
+GLIBC_2.41 _IO_list_lock F
+GLIBC_2.41 _IO_list_resetlock F
+GLIBC_2.41 _IO_list_unlock F
+GLIBC_2.41 _IO_marker_delta F
+GLIBC_2.41 _IO_marker_difference F
+GLIBC_2.41 _IO_padn F
+GLIBC_2.41 _IO_peekc_locked F
+GLIBC_2.41 _IO_popen F
+GLIBC_2.41 _IO_printf F
+GLIBC_2.41 _IO_proc_close F
+GLIBC_2.41 _IO_proc_open F
+GLIBC_2.41 _IO_putc F
+GLIBC_2.41 _IO_puts F
+GLIBC_2.41 _IO_remove_marker F
+GLIBC_2.41 _IO_seekmark F
+GLIBC_2.41 _IO_seekoff F
+GLIBC_2.41 _IO_seekpos F
+GLIBC_2.41 _IO_seekwmark F
+GLIBC_2.41 _IO_setb F
+GLIBC_2.41 _IO_setbuffer F
+GLIBC_2.41 _IO_setvbuf F
+GLIBC_2.41 _IO_sgetn F
+GLIBC_2.41 _IO_sprintf F
+GLIBC_2.41 _IO_sputbackc F
+GLIBC_2.41 _IO_sputbackwc F
+GLIBC_2.41 _IO_sscanf F
+GLIBC_2.41 _IO_str_init_readonly F
+GLIBC_2.41 _IO_str_init_static F
+GLIBC_2.41 _IO_str_overflow F
+GLIBC_2.41 _IO_str_pbackfail F
+GLIBC_2.41 _IO_str_seekoff F
+GLIBC_2.41 _IO_str_underflow F
+GLIBC_2.41 _IO_sungetc F
+GLIBC_2.41 _IO_sungetwc F
+GLIBC_2.41 _IO_switch_to_get_mode F
+GLIBC_2.41 _IO_switch_to_main_wget_area F
+GLIBC_2.41 _IO_switch_to_wbackup_area F
+GLIBC_2.41 _IO_switch_to_wget_mode F
+GLIBC_2.41 _IO_un_link F
+GLIBC_2.41 _IO_ungetc F
+GLIBC_2.41 _IO_unsave_markers F
+GLIBC_2.41 _IO_unsave_wmarkers F
+GLIBC_2.41 _IO_vfprintf F
+GLIBC_2.41 _IO_vsprintf F
+GLIBC_2.41 _IO_wdefault_doallocate F
+GLIBC_2.41 _IO_wdefault_finish F
+GLIBC_2.41 _IO_wdefault_pbackfail F
+GLIBC_2.41 _IO_wdefault_uflow F
+GLIBC_2.41 _IO_wdefault_xsgetn F
+GLIBC_2.41 _IO_wdefault_xsputn F
+GLIBC_2.41 _IO_wdo_write F
+GLIBC_2.41 _IO_wdoallocbuf F
+GLIBC_2.41 _IO_wfile_jumps D 0x54
+GLIBC_2.41 _IO_wfile_overflow F
+GLIBC_2.41 _IO_wfile_seekoff F
+GLIBC_2.41 _IO_wfile_sync F
+GLIBC_2.41 _IO_wfile_underflow F
+GLIBC_2.41 _IO_wfile_xsputn F
+GLIBC_2.41 _IO_wmarker_delta F
+GLIBC_2.41 _IO_wsetb F
+GLIBC_2.41 ___brk_addr D 0x4
+GLIBC_2.41 __adjtimex F
+GLIBC_2.41 __argz_count F
+GLIBC_2.41 __argz_next F
+GLIBC_2.41 __argz_stringify F
+GLIBC_2.41 __asprintf F
+GLIBC_2.41 __asprintf_chk F
+GLIBC_2.41 __assert F
+GLIBC_2.41 __assert_fail F
+GLIBC_2.41 __assert_perror_fail F
+GLIBC_2.41 __backtrace F
+GLIBC_2.41 __backtrace_symbols F
+GLIBC_2.41 __backtrace_symbols_fd F
+GLIBC_2.41 __bsd_getpgrp F
+GLIBC_2.41 __bzero F
+GLIBC_2.41 __check_rhosts_file D 0x4
+GLIBC_2.41 __chk_fail F
+GLIBC_2.41 __clone F
+GLIBC_2.41 __close F
+GLIBC_2.41 __cmsg_nxthdr F
+GLIBC_2.41 __confstr_chk F
+GLIBC_2.41 __connect F
+GLIBC_2.41 __ctype_b_loc F
+GLIBC_2.41 __ctype_get_mb_cur_max F
+GLIBC_2.41 __ctype_tolower_loc F
+GLIBC_2.41 __ctype_toupper_loc F
+GLIBC_2.41 __curbrk D 0x4
+GLIBC_2.41 __cxa_at_quick_exit F
+GLIBC_2.41 __cxa_atexit F
+GLIBC_2.41 __cxa_finalize F
+GLIBC_2.41 __cxa_thread_atexit_impl F
+GLIBC_2.41 __cyg_profile_func_enter F
+GLIBC_2.41 __cyg_profile_func_exit F
+GLIBC_2.41 __daylight D 0x4
+GLIBC_2.41 __dcgettext F
+GLIBC_2.41 __dgettext F
+GLIBC_2.41 __dprintf_chk F
+GLIBC_2.41 __dup2 F
+GLIBC_2.41 __duplocale F
+GLIBC_2.41 __endmntent F
+GLIBC_2.41 __environ D 0x4
+GLIBC_2.41 __errno_location F
+GLIBC_2.41 __explicit_bzero_chk F
+GLIBC_2.41 __fbufsize F
+GLIBC_2.41 __fcntl F
+GLIBC_2.41 __fdelt_chk F
+GLIBC_2.41 __fdelt_warn F
+GLIBC_2.41 __ffs F
+GLIBC_2.41 __fgets_chk F
+GLIBC_2.41 __fgets_unlocked_chk F
+GLIBC_2.41 __fgetws_chk F
+GLIBC_2.41 __fgetws_unlocked_chk F
+GLIBC_2.41 __finite F
+GLIBC_2.41 __finitef F
+GLIBC_2.41 __finitel F
+GLIBC_2.41 __flbf F
+GLIBC_2.41 __fork F
+GLIBC_2.41 __fpending F
+GLIBC_2.41 __fprintf_chk F
+GLIBC_2.41 __fpu_control D 0x4
+GLIBC_2.41 __fpurge F
+GLIBC_2.41 __fread_chk F
+GLIBC_2.41 __fread_unlocked_chk F
+GLIBC_2.41 __freadable F
+GLIBC_2.41 __freading F
+GLIBC_2.41 __freelocale F
+GLIBC_2.41 __fsetlocking F
+GLIBC_2.41 __fwprintf_chk F
+GLIBC_2.41 __fwritable F
+GLIBC_2.41 __fwriting F
+GLIBC_2.41 __getauxval F
+GLIBC_2.41 __getcwd_chk F
+GLIBC_2.41 __getdelim F
+GLIBC_2.41 __getdomainname_chk F
+GLIBC_2.41 __getgroups_chk F
+GLIBC_2.41 __gethostname_chk F
+GLIBC_2.41 __getlogin_r_chk F
+GLIBC_2.41 __getmntent_r F
+GLIBC_2.41 __getpagesize F
+GLIBC_2.41 __getpgid F
+GLIBC_2.41 __getpid F
+GLIBC_2.41 __gets_chk F
+GLIBC_2.41 __gettimeofday F
+GLIBC_2.41 __getwd_chk F
+GLIBC_2.41 __gmtime_r F
+GLIBC_2.41 __h_errno_location F
+GLIBC_2.41 __isalnum_l F
+GLIBC_2.41 __isalpha_l F
+GLIBC_2.41 __isascii_l F
+GLIBC_2.41 __isblank_l F
+GLIBC_2.41 __iscntrl_l F
+GLIBC_2.41 __isctype F
+GLIBC_2.41 __isdigit_l F
+GLIBC_2.41 __isgraph_l F
+GLIBC_2.41 __isinf F
+GLIBC_2.41 __isinff F
+GLIBC_2.41 __isinfl F
+GLIBC_2.41 __islower_l F
+GLIBC_2.41 __isnan F
+GLIBC_2.41 __isnanf F
+GLIBC_2.41 __isnanl F
+GLIBC_2.41 __isoc23_fscanf F
+GLIBC_2.41 __isoc23_fwscanf F
+GLIBC_2.41 __isoc23_scanf F
+GLIBC_2.41 __isoc23_sscanf F
+GLIBC_2.41 __isoc23_strtoimax F
+GLIBC_2.41 __isoc23_strtol F
+GLIBC_2.41 __isoc23_strtol_l F
+GLIBC_2.41 __isoc23_strtoll F
+GLIBC_2.41 __isoc23_strtoll_l F
+GLIBC_2.41 __isoc23_strtoul F
+GLIBC_2.41 __isoc23_strtoul_l F
+GLIBC_2.41 __isoc23_strtoull F
+GLIBC_2.41 __isoc23_strtoull_l F
+GLIBC_2.41 __isoc23_strtoumax F
+GLIBC_2.41 __isoc23_swscanf F
+GLIBC_2.41 __isoc23_vfscanf F
+GLIBC_2.41 __isoc23_vfwscanf F
+GLIBC_2.41 __isoc23_vscanf F
+GLIBC_2.41 __isoc23_vsscanf F
+GLIBC_2.41 __isoc23_vswscanf F
+GLIBC_2.41 __isoc23_vwscanf F
+GLIBC_2.41 __isoc23_wcstoimax F
+GLIBC_2.41 __isoc23_wcstol F
+GLIBC_2.41 __isoc23_wcstol_l F
+GLIBC_2.41 __isoc23_wcstoll F
+GLIBC_2.41 __isoc23_wcstoll_l F
+GLIBC_2.41 __isoc23_wcstoul F
+GLIBC_2.41 __isoc23_wcstoul_l F
+GLIBC_2.41 __isoc23_wcstoull F
+GLIBC_2.41 __isoc23_wcstoull_l F
+GLIBC_2.41 __isoc23_wcstoumax F
+GLIBC_2.41 __isoc23_wscanf F
+GLIBC_2.41 __isoc99_fscanf F
+GLIBC_2.41 __isoc99_fwscanf F
+GLIBC_2.41 __isoc99_scanf F
+GLIBC_2.41 __isoc99_sscanf F
+GLIBC_2.41 __isoc99_swscanf F
+GLIBC_2.41 __isoc99_vfscanf F
+GLIBC_2.41 __isoc99_vfwscanf F
+GLIBC_2.41 __isoc99_vscanf F
+GLIBC_2.41 __isoc99_vsscanf F
+GLIBC_2.41 __isoc99_vswscanf F
+GLIBC_2.41 __isoc99_vwscanf F
+GLIBC_2.41 __isoc99_wscanf F
+GLIBC_2.41 __isprint_l F
+GLIBC_2.41 __ispunct_l F
+GLIBC_2.41 __isspace_l F
+GLIBC_2.41 __isupper_l F
+GLIBC_2.41 __iswalnum_l F
+GLIBC_2.41 __iswalpha_l F
+GLIBC_2.41 __iswblank_l F
+GLIBC_2.41 __iswcntrl_l F
+GLIBC_2.41 __iswctype F
+GLIBC_2.41 __iswctype_l F
+GLIBC_2.41 __iswdigit_l F
+GLIBC_2.41 __iswgraph_l F
+GLIBC_2.41 __iswlower_l F
+GLIBC_2.41 __iswprint_l F
+GLIBC_2.41 __iswpunct_l F
+GLIBC_2.41 __iswspace_l F
+GLIBC_2.41 __iswupper_l F
+GLIBC_2.41 __iswxdigit_l F
+GLIBC_2.41 __isxdigit_l F
+GLIBC_2.41 __libc_allocate_rtsig F
+GLIBC_2.41 __libc_calloc F
+GLIBC_2.41 __libc_current_sigrtmax F
+GLIBC_2.41 __libc_current_sigrtmin F
+GLIBC_2.41 __libc_free F
+GLIBC_2.41 __libc_freeres F
+GLIBC_2.41 __libc_init_first F
+GLIBC_2.41 __libc_mallinfo F
+GLIBC_2.41 __libc_malloc F
+GLIBC_2.41 __libc_mallopt F
+GLIBC_2.41 __libc_memalign F
+GLIBC_2.41 __libc_pvalloc F
+GLIBC_2.41 __libc_realloc F
+GLIBC_2.41 __libc_sa_len F
+GLIBC_2.41 __libc_single_threaded D 0x1
+GLIBC_2.41 __libc_start_main F
+GLIBC_2.41 __libc_valloc F
+GLIBC_2.41 __longjmp_chk F
+GLIBC_2.41 __lseek F
+GLIBC_2.41 __mbrlen F
+GLIBC_2.41 __mbrtowc F
+GLIBC_2.41 __mbsnrtowcs_chk F
+GLIBC_2.41 __mbsrtowcs_chk F
+GLIBC_2.41 __mbstowcs_chk F
+GLIBC_2.41 __memcmpeq F
+GLIBC_2.41 __memcpy_chk F
+GLIBC_2.41 __memmove_chk F
+GLIBC_2.41 __mempcpy F
+GLIBC_2.41 __mempcpy_chk F
+GLIBC_2.41 __memset_chk F
+GLIBC_2.41 __monstartup F
+GLIBC_2.41 __mq_open_2 F
+GLIBC_2.41 __nanosleep F
+GLIBC_2.41 __newlocale F
+GLIBC_2.41 __nl_langinfo_l F
+GLIBC_2.41 __nss_configure_lookup F
+GLIBC_2.41 __nss_hostname_digits_dots F
+GLIBC_2.41 __obstack_printf_chk F
+GLIBC_2.41 __obstack_vprintf_chk F
+GLIBC_2.41 __open F
+GLIBC_2.41 __open64 F
+GLIBC_2.41 __open64_2 F
+GLIBC_2.41 __open_2 F
+GLIBC_2.41 __openat64_2 F
+GLIBC_2.41 __openat_2 F
+GLIBC_2.41 __overflow F
+GLIBC_2.41 __pipe F
+GLIBC_2.41 __poll F
+GLIBC_2.41 __poll_chk F
+GLIBC_2.41 __posix_getopt F
+GLIBC_2.41 __ppoll_chk F
+GLIBC_2.41 __pread64 F
+GLIBC_2.41 __pread64_chk F
+GLIBC_2.41 __pread_chk F
+GLIBC_2.41 __printf_chk F
+GLIBC_2.41 __printf_fp F
+GLIBC_2.41 __profile_frequency F
+GLIBC_2.41 __progname D 0x4
+GLIBC_2.41 __progname_full D 0x4
+GLIBC_2.41 __pthread_cleanup_routine F
+GLIBC_2.41 __pthread_key_create F
+GLIBC_2.41 __pthread_register_cancel F
+GLIBC_2.41 __pthread_register_cancel_defer F
+GLIBC_2.41 __pthread_rwlock_unlock F
+GLIBC_2.41 __pthread_unregister_cancel F
+GLIBC_2.41 __pthread_unregister_cancel_restore F
+GLIBC_2.41 __pthread_unwind_next F
+GLIBC_2.41 __ptsname_r_chk F
+GLIBC_2.41 __pwrite64 F
+GLIBC_2.41 __rawmemchr F
+GLIBC_2.41 __rcmd_errstr D 0x4
+GLIBC_2.41 __read F
+GLIBC_2.41 __read_chk F
+GLIBC_2.41 __readlink_chk F
+GLIBC_2.41 __readlinkat_chk F
+GLIBC_2.41 __realpath_chk F
+GLIBC_2.41 __recv_chk F
+GLIBC_2.41 __recvfrom_chk F
+GLIBC_2.41 __register_atfork F
+GLIBC_2.41 __res_init F
+GLIBC_2.41 __res_nclose F
+GLIBC_2.41 __res_ninit F
+GLIBC_2.41 __res_randomid F
+GLIBC_2.41 __res_state F
+GLIBC_2.41 __sbrk F
+GLIBC_2.41 __sched_cpualloc F
+GLIBC_2.41 __sched_cpucount F
+GLIBC_2.41 __sched_cpufree F
+GLIBC_2.41 __sched_get_priority_max F
+GLIBC_2.41 __sched_get_priority_min F
+GLIBC_2.41 __sched_getparam F
+GLIBC_2.41 __sched_getscheduler F
+GLIBC_2.41 __sched_setscheduler F
+GLIBC_2.41 __sched_yield F
+GLIBC_2.41 __select F
+GLIBC_2.41 __send F
+GLIBC_2.41 __setmntent F
+GLIBC_2.41 __setpgid F
+GLIBC_2.41 __sigaction F
+GLIBC_2.41 __signbit F
+GLIBC_2.41 __signbitf F
+GLIBC_2.41 __signbitl F
+GLIBC_2.41 __sigpause F
+GLIBC_2.41 __sigsetjmp F
+GLIBC_2.41 __sigsuspend F
+GLIBC_2.41 __snprintf_chk F
+GLIBC_2.41 __sprintf_chk F
+GLIBC_2.41 __stack_chk_fail F
+GLIBC_2.41 __statfs F
+GLIBC_2.41 __stpcpy F
+GLIBC_2.41 __stpcpy_chk F
+GLIBC_2.41 __stpncpy F
+GLIBC_2.41 __stpncpy_chk F
+GLIBC_2.41 __strcasecmp F
+GLIBC_2.41 __strcasecmp_l F
+GLIBC_2.41 __strcasestr F
+GLIBC_2.41 __strcat_chk F
+GLIBC_2.41 __strcoll_l F
+GLIBC_2.41 __strcpy_chk F
+GLIBC_2.41 __strdup F
+GLIBC_2.41 __strerror_r F
+GLIBC_2.41 __strfmon_l F
+GLIBC_2.41 __strftime_l F
+GLIBC_2.41 __strlcat_chk F
+GLIBC_2.41 __strlcpy_chk F
+GLIBC_2.41 __strncasecmp_l F
+GLIBC_2.41 __strncat_chk F
+GLIBC_2.41 __strncpy_chk F
+GLIBC_2.41 __strndup F
+GLIBC_2.41 __strsep_g F
+GLIBC_2.41 __strtod_internal F
+GLIBC_2.41 __strtod_l F
+GLIBC_2.41 __strtof_internal F
+GLIBC_2.41 __strtof_l F
+GLIBC_2.41 __strtok_r F
+GLIBC_2.41 __strtol_internal F
+GLIBC_2.41 __strtol_l F
+GLIBC_2.41 __strtold_internal F
+GLIBC_2.41 __strtold_l F
+GLIBC_2.41 __strtoll_internal F
+GLIBC_2.41 __strtoll_l F
+GLIBC_2.41 __strtoul_internal F
+GLIBC_2.41 __strtoul_l F
+GLIBC_2.41 __strtoull_internal F
+GLIBC_2.41 __strtoull_l F
+GLIBC_2.41 __strverscmp F
+GLIBC_2.41 __strxfrm_l F
+GLIBC_2.41 __swprintf_chk F
+GLIBC_2.41 __sysconf F
+GLIBC_2.41 __syslog_chk F
+GLIBC_2.41 __sysv_signal F
+GLIBC_2.41 __timezone D 0x4
+GLIBC_2.41 __toascii_l F
+GLIBC_2.41 __tolower_l F
+GLIBC_2.41 __toupper_l F
+GLIBC_2.41 __towctrans F
+GLIBC_2.41 __towctrans_l F
+GLIBC_2.41 __towlower_l F
+GLIBC_2.41 __towupper_l F
+GLIBC_2.41 __ttyname_r_chk F
+GLIBC_2.41 __tzname D 0x8
+GLIBC_2.41 __uflow F
+GLIBC_2.41 __underflow F
+GLIBC_2.41 __uselocale F
+GLIBC_2.41 __vasprintf_chk F
+GLIBC_2.41 __vdprintf_chk F
+GLIBC_2.41 __vfork F
+GLIBC_2.41 __vfprintf_chk F
+GLIBC_2.41 __vfscanf F
+GLIBC_2.41 __vfwprintf_chk F
+GLIBC_2.41 __vprintf_chk F
+GLIBC_2.41 __vsnprintf F
+GLIBC_2.41 __vsnprintf_chk F
+GLIBC_2.41 __vsprintf_chk F
+GLIBC_2.41 __vsscanf F
+GLIBC_2.41 __vswprintf_chk F
+GLIBC_2.41 __vsyslog_chk F
+GLIBC_2.41 __vwprintf_chk F
+GLIBC_2.41 __wait F
+GLIBC_2.41 __waitpid F
+GLIBC_2.41 __wcpcpy_chk F
+GLIBC_2.41 __wcpncpy_chk F
+GLIBC_2.41 __wcrtomb_chk F
+GLIBC_2.41 __wcscasecmp_l F
+GLIBC_2.41 __wcscat_chk F
+GLIBC_2.41 __wcscoll_l F
+GLIBC_2.41 __wcscpy_chk F
+GLIBC_2.41 __wcsftime_l F
+GLIBC_2.41 __wcslcat_chk F
+GLIBC_2.41 __wcslcpy_chk F
+GLIBC_2.41 __wcsncasecmp_l F
+GLIBC_2.41 __wcsncat_chk F
+GLIBC_2.41 __wcsncpy_chk F
+GLIBC_2.41 __wcsnrtombs_chk F
+GLIBC_2.41 __wcsrtombs_chk F
+GLIBC_2.41 __wcstod_internal F
+GLIBC_2.41 __wcstod_l F
+GLIBC_2.41 __wcstof_internal F
+GLIBC_2.41 __wcstof_l F
+GLIBC_2.41 __wcstol_internal F
+GLIBC_2.41 __wcstol_l F
+GLIBC_2.41 __wcstold_internal F
+GLIBC_2.41 __wcstold_l F
+GLIBC_2.41 __wcstoll_internal F
+GLIBC_2.41 __wcstoll_l F
+GLIBC_2.41 __wcstombs_chk F
+GLIBC_2.41 __wcstoul_internal F
+GLIBC_2.41 __wcstoul_l F
+GLIBC_2.41 __wcstoull_internal F
+GLIBC_2.41 __wcstoull_l F
+GLIBC_2.41 __wcsxfrm_l F
+GLIBC_2.41 __wctomb_chk F
+GLIBC_2.41 __wctrans_l F
+GLIBC_2.41 __wctype_l F
+GLIBC_2.41 __wmemcpy_chk F
+GLIBC_2.41 __wmemmove_chk F
+GLIBC_2.41 __wmempcpy_chk F
+GLIBC_2.41 __wmemset_chk F
+GLIBC_2.41 __woverflow F
+GLIBC_2.41 __wprintf_chk F
+GLIBC_2.41 __write F
+GLIBC_2.41 __wuflow F
+GLIBC_2.41 __wunderflow F
+GLIBC_2.41 __xpg_basename F
+GLIBC_2.41 __xpg_sigpause F
+GLIBC_2.41 __xpg_strerror_r F
+GLIBC_2.41 _dl_find_object F
+GLIBC_2.41 _dl_mcount_wrapper_check F
+GLIBC_2.41 _environ D 0x4
+GLIBC_2.41 _exit F
+GLIBC_2.41 _flushlbf F
+GLIBC_2.41 _libc_intl_domainname D 0x5
+GLIBC_2.41 _longjmp F
+GLIBC_2.41 _mcleanup F
+GLIBC_2.41 _mcount F
+GLIBC_2.41 _nl_default_dirname D 0x12
+GLIBC_2.41 _nl_domain_bindings D 0x4
+GLIBC_2.41 _nl_msg_cat_cntr D 0x4
+GLIBC_2.41 _obstack_allocated_p F
+GLIBC_2.41 _obstack_begin F
+GLIBC_2.41 _obstack_begin_1 F
+GLIBC_2.41 _obstack_free F
+GLIBC_2.41 _obstack_memory_used F
+GLIBC_2.41 _obstack_newchunk F
+GLIBC_2.41 _pthread_cleanup_pop F
+GLIBC_2.41 _pthread_cleanup_push F
+GLIBC_2.41 _res D 0x200
+GLIBC_2.41 _res_hconf D 0x30
+GLIBC_2.41 _setjmp F
+GLIBC_2.41 _tolower F
+GLIBC_2.41 _toupper F
+GLIBC_2.41 a64l F
+GLIBC_2.41 abort F
+GLIBC_2.41 abs F
+GLIBC_2.41 accept F
+GLIBC_2.41 accept4 F
+GLIBC_2.41 access F
+GLIBC_2.41 acct F
+GLIBC_2.41 addmntent F
+GLIBC_2.41 addseverity F
+GLIBC_2.41 adjtime F
+GLIBC_2.41 adjtimex F
+GLIBC_2.41 aio_cancel F
+GLIBC_2.41 aio_cancel64 F
+GLIBC_2.41 aio_error F
+GLIBC_2.41 aio_error64 F
+GLIBC_2.41 aio_fsync F
+GLIBC_2.41 aio_fsync64 F
+GLIBC_2.41 aio_init F
+GLIBC_2.41 aio_read F
+GLIBC_2.41 aio_read64 F
+GLIBC_2.41 aio_return F
+GLIBC_2.41 aio_return64 F
+GLIBC_2.41 aio_suspend F
+GLIBC_2.41 aio_suspend64 F
+GLIBC_2.41 aio_write F
+GLIBC_2.41 aio_write64 F
+GLIBC_2.41 alarm F
+GLIBC_2.41 aligned_alloc F
+GLIBC_2.41 alphasort F
+GLIBC_2.41 alphasort64 F
+GLIBC_2.41 arc4random F
+GLIBC_2.41 arc4random_buf F
+GLIBC_2.41 arc4random_uniform F
+GLIBC_2.41 argp_err_exit_status D 0x4
+GLIBC_2.41 argp_error F
+GLIBC_2.41 argp_failure F
+GLIBC_2.41 argp_help F
+GLIBC_2.41 argp_parse F
+GLIBC_2.41 argp_program_bug_address D 0x4
+GLIBC_2.41 argp_program_version D 0x4
+GLIBC_2.41 argp_program_version_hook D 0x4
+GLIBC_2.41 argp_state_help F
+GLIBC_2.41 argp_usage F
+GLIBC_2.41 argz_add F
+GLIBC_2.41 argz_add_sep F
+GLIBC_2.41 argz_append F
+GLIBC_2.41 argz_count F
+GLIBC_2.41 argz_create F
+GLIBC_2.41 argz_create_sep F
+GLIBC_2.41 argz_delete F
+GLIBC_2.41 argz_extract F
+GLIBC_2.41 argz_insert F
+GLIBC_2.41 argz_next F
+GLIBC_2.41 argz_replace F
+GLIBC_2.41 argz_stringify F
+GLIBC_2.41 asctime F
+GLIBC_2.41 asctime_r F
+GLIBC_2.41 asprintf F
+GLIBC_2.41 atof F
+GLIBC_2.41 atoi F
+GLIBC_2.41 atol F
+GLIBC_2.41 atoll F
+GLIBC_2.41 backtrace F
+GLIBC_2.41 backtrace_symbols F
+GLIBC_2.41 backtrace_symbols_fd F
+GLIBC_2.41 basename F
+GLIBC_2.41 bcmp F
+GLIBC_2.41 bcopy F
+GLIBC_2.41 bind F
+GLIBC_2.41 bind_textdomain_codeset F
+GLIBC_2.41 bindresvport F
+GLIBC_2.41 bindtextdomain F
+GLIBC_2.41 brk F
+GLIBC_2.41 bsd_signal F
+GLIBC_2.41 bsearch F
+GLIBC_2.41 btowc F
+GLIBC_2.41 bzero F
+GLIBC_2.41 c16rtomb F
+GLIBC_2.41 c32rtomb F
+GLIBC_2.41 c8rtomb F
+GLIBC_2.41 call_once F
+GLIBC_2.41 calloc F
+GLIBC_2.41 canonicalize_file_name F
+GLIBC_2.41 capget F
+GLIBC_2.41 capset F
+GLIBC_2.41 catclose F
+GLIBC_2.41 catgets F
+GLIBC_2.41 catopen F
+GLIBC_2.41 cfgetispeed F
+GLIBC_2.41 cfgetospeed F
+GLIBC_2.41 cfmakeraw F
+GLIBC_2.41 cfsetispeed F
+GLIBC_2.41 cfsetospeed F
+GLIBC_2.41 cfsetspeed F
+GLIBC_2.41 chdir F
+GLIBC_2.41 chflags F
+GLIBC_2.41 chmod F
+GLIBC_2.41 chown F
+GLIBC_2.41 chroot F
+GLIBC_2.41 clearenv F
+GLIBC_2.41 clearerr F
+GLIBC_2.41 clearerr_unlocked F
+GLIBC_2.41 clock F
+GLIBC_2.41 clock_adjtime F
+GLIBC_2.41 clock_getcpuclockid F
+GLIBC_2.41 clock_getres F
+GLIBC_2.41 clock_gettime F
+GLIBC_2.41 clock_nanosleep F
+GLIBC_2.41 clock_settime F
+GLIBC_2.41 clone F
+GLIBC_2.41 close F
+GLIBC_2.41 close_range F
+GLIBC_2.41 closedir F
+GLIBC_2.41 closefrom F
+GLIBC_2.41 closelog F
+GLIBC_2.41 cnd_broadcast F
+GLIBC_2.41 cnd_destroy F
+GLIBC_2.41 cnd_init F
+GLIBC_2.41 cnd_signal F
+GLIBC_2.41 cnd_timedwait F
+GLIBC_2.41 cnd_wait F
+GLIBC_2.41 confstr F
+GLIBC_2.41 connect F
+GLIBC_2.41 copy_file_range F
+GLIBC_2.41 copysign F
+GLIBC_2.41 copysignf F
+GLIBC_2.41 copysignl F
+GLIBC_2.41 creat F
+GLIBC_2.41 creat64 F
+GLIBC_2.41 ctermid F
+GLIBC_2.41 ctime F
+GLIBC_2.41 ctime_r F
+GLIBC_2.41 cuserid F
+GLIBC_2.41 daemon F
+GLIBC_2.41 daylight D 0x4
+GLIBC_2.41 dcgettext F
+GLIBC_2.41 dcngettext F
+GLIBC_2.41 delete_module F
+GLIBC_2.41 dgettext F
+GLIBC_2.41 difftime F
+GLIBC_2.41 dirfd F
+GLIBC_2.41 dirname F
+GLIBC_2.41 div F
+GLIBC_2.41 dl_iterate_phdr F
+GLIBC_2.41 dladdr F
+GLIBC_2.41 dladdr1 F
+GLIBC_2.41 dlclose F
+GLIBC_2.41 dlerror F
+GLIBC_2.41 dlinfo F
+GLIBC_2.41 dlmopen F
+GLIBC_2.41 dlopen F
+GLIBC_2.41 dlsym F
+GLIBC_2.41 dlvsym F
+GLIBC_2.41 dn_comp F
+GLIBC_2.41 dn_expand F
+GLIBC_2.41 dn_skipname F
+GLIBC_2.41 dngettext F
+GLIBC_2.41 dprintf F
+GLIBC_2.41 drand48 F
+GLIBC_2.41 drand48_r F
+GLIBC_2.41 dup F
+GLIBC_2.41 dup2 F
+GLIBC_2.41 dup3 F
+GLIBC_2.41 duplocale F
+GLIBC_2.41 dysize F
+GLIBC_2.41 eaccess F
+GLIBC_2.41 ecvt F
+GLIBC_2.41 ecvt_r F
+GLIBC_2.41 endaliasent F
+GLIBC_2.41 endfsent F
+GLIBC_2.41 endgrent F
+GLIBC_2.41 endhostent F
+GLIBC_2.41 endmntent F
+GLIBC_2.41 endnetent F
+GLIBC_2.41 endnetgrent F
+GLIBC_2.41 endprotoent F
+GLIBC_2.41 endpwent F
+GLIBC_2.41 endrpcent F
+GLIBC_2.41 endservent F
+GLIBC_2.41 endsgent F
+GLIBC_2.41 endspent F
+GLIBC_2.41 endttyent F
+GLIBC_2.41 endusershell F
+GLIBC_2.41 endutent F
+GLIBC_2.41 endutxent F
+GLIBC_2.41 environ D 0x4
+GLIBC_2.41 envz_add F
+GLIBC_2.41 envz_entry F
+GLIBC_2.41 envz_get F
+GLIBC_2.41 envz_merge F
+GLIBC_2.41 envz_remove F
+GLIBC_2.41 envz_strip F
+GLIBC_2.41 epoll_create F
+GLIBC_2.41 epoll_create1 F
+GLIBC_2.41 epoll_ctl F
+GLIBC_2.41 epoll_pwait F
+GLIBC_2.41 epoll_pwait2 F
+GLIBC_2.41 epoll_wait F
+GLIBC_2.41 erand48 F
+GLIBC_2.41 erand48_r F
+GLIBC_2.41 err F
+GLIBC_2.41 error F
+GLIBC_2.41 error_at_line F
+GLIBC_2.41 error_message_count D 0x4
+GLIBC_2.41 error_one_per_line D 0x4
+GLIBC_2.41 error_print_progname D 0x4
+GLIBC_2.41 errx F
+GLIBC_2.41 ether_aton F
+GLIBC_2.41 ether_aton_r F
+GLIBC_2.41 ether_hostton F
+GLIBC_2.41 ether_line F
+GLIBC_2.41 ether_ntoa F
+GLIBC_2.41 ether_ntoa_r F
+GLIBC_2.41 ether_ntohost F
+GLIBC_2.41 euidaccess F
+GLIBC_2.41 eventfd F
+GLIBC_2.41 eventfd_read F
+GLIBC_2.41 eventfd_write F
+GLIBC_2.41 execl F
+GLIBC_2.41 execle F
+GLIBC_2.41 execlp F
+GLIBC_2.41 execv F
+GLIBC_2.41 execve F
+GLIBC_2.41 execveat F
+GLIBC_2.41 execvp F
+GLIBC_2.41 execvpe F
+GLIBC_2.41 exit F
+GLIBC_2.41 explicit_bzero F
+GLIBC_2.41 faccessat F
+GLIBC_2.41 fallocate F
+GLIBC_2.41 fallocate64 F
+GLIBC_2.41 fanotify_init F
+GLIBC_2.41 fanotify_mark F
+GLIBC_2.41 fchdir F
+GLIBC_2.41 fchflags F
+GLIBC_2.41 fchmod F
+GLIBC_2.41 fchmodat F
+GLIBC_2.41 fchown F
+GLIBC_2.41 fchownat F
+GLIBC_2.41 fclose F
+GLIBC_2.41 fcloseall F
+GLIBC_2.41 fcntl F
+GLIBC_2.41 fcntl64 F
+GLIBC_2.41 fcvt F
+GLIBC_2.41 fcvt_r F
+GLIBC_2.41 fdatasync F
+GLIBC_2.41 fdopen F
+GLIBC_2.41 fdopendir F
+GLIBC_2.41 feof F
+GLIBC_2.41 feof_unlocked F
+GLIBC_2.41 ferror F
+GLIBC_2.41 ferror_unlocked F
+GLIBC_2.41 fexecve F
+GLIBC_2.41 fflush F
+GLIBC_2.41 fflush_unlocked F
+GLIBC_2.41 ffs F
+GLIBC_2.41 ffsl F
+GLIBC_2.41 ffsll F
+GLIBC_2.41 fgetc F
+GLIBC_2.41 fgetc_unlocked F
+GLIBC_2.41 fgetgrent F
+GLIBC_2.41 fgetgrent_r F
+GLIBC_2.41 fgetpos F
+GLIBC_2.41 fgetpos64 F
+GLIBC_2.41 fgetpwent F
+GLIBC_2.41 fgetpwent_r F
+GLIBC_2.41 fgets F
+GLIBC_2.41 fgets_unlocked F
+GLIBC_2.41 fgetsgent F
+GLIBC_2.41 fgetsgent_r F
+GLIBC_2.41 fgetspent F
+GLIBC_2.41 fgetspent_r F
+GLIBC_2.41 fgetwc F
+GLIBC_2.41 fgetwc_unlocked F
+GLIBC_2.41 fgetws F
+GLIBC_2.41 fgetws_unlocked F
+GLIBC_2.41 fgetxattr F
+GLIBC_2.41 fileno F
+GLIBC_2.41 fileno_unlocked F
+GLIBC_2.41 finite F
+GLIBC_2.41 finitef F
+GLIBC_2.41 finitel F
+GLIBC_2.41 flistxattr F
+GLIBC_2.41 flock F
+GLIBC_2.41 flockfile F
+GLIBC_2.41 fmemopen F
+GLIBC_2.41 fmtmsg F
+GLIBC_2.41 fnmatch F
+GLIBC_2.41 fopen F
+GLIBC_2.41 fopen64 F
+GLIBC_2.41 fopencookie F
+GLIBC_2.41 fork F
+GLIBC_2.41 forkpty F
+GLIBC_2.41 fpathconf F
+GLIBC_2.41 fprintf F
+GLIBC_2.41 fputc F
+GLIBC_2.41 fputc_unlocked F
+GLIBC_2.41 fputs F
+GLIBC_2.41 fputs_unlocked F
+GLIBC_2.41 fputwc F
+GLIBC_2.41 fputwc_unlocked F
+GLIBC_2.41 fputws F
+GLIBC_2.41 fputws_unlocked F
+GLIBC_2.41 fread F
+GLIBC_2.41 fread_unlocked F
+GLIBC_2.41 free F
+GLIBC_2.41 freeaddrinfo F
+GLIBC_2.41 freeifaddrs F
+GLIBC_2.41 freelocale F
+GLIBC_2.41 fremovexattr F
+GLIBC_2.41 freopen F
+GLIBC_2.41 freopen64 F
+GLIBC_2.41 frexp F
+GLIBC_2.41 frexpf F
+GLIBC_2.41 frexpl F
+GLIBC_2.41 fscanf F
+GLIBC_2.41 fsconfig F
+GLIBC_2.41 fseek F
+GLIBC_2.41 fseeko F
+GLIBC_2.41 fseeko64 F
+GLIBC_2.41 fsetpos F
+GLIBC_2.41 fsetpos64 F
+GLIBC_2.41 fsetxattr F
+GLIBC_2.41 fsmount F
+GLIBC_2.41 fsopen F
+GLIBC_2.41 fspick F
+GLIBC_2.41 fstat F
+GLIBC_2.41 fstat64 F
+GLIBC_2.41 fstatat F
+GLIBC_2.41 fstatat64 F
+GLIBC_2.41 fstatfs F
+GLIBC_2.41 fstatfs64 F
+GLIBC_2.41 fstatvfs F
+GLIBC_2.41 fstatvfs64 F
+GLIBC_2.41 fsync F
+GLIBC_2.41 ftell F
+GLIBC_2.41 ftello F
+GLIBC_2.41 ftello64 F
+GLIBC_2.41 ftime F
+GLIBC_2.41 ftok F
+GLIBC_2.41 ftruncate F
+GLIBC_2.41 ftruncate64 F
+GLIBC_2.41 ftrylockfile F
+GLIBC_2.41 fts64_children F
+GLIBC_2.41 fts64_close F
+GLIBC_2.41 fts64_open F
+GLIBC_2.41 fts64_read F
+GLIBC_2.41 fts64_set F
+GLIBC_2.41 fts_children F
+GLIBC_2.41 fts_close F
+GLIBC_2.41 fts_open F
+GLIBC_2.41 fts_read F
+GLIBC_2.41 fts_set F
+GLIBC_2.41 ftw F
+GLIBC_2.41 ftw64 F
+GLIBC_2.41 funlockfile F
+GLIBC_2.41 futimens F
+GLIBC_2.41 futimes F
+GLIBC_2.41 futimesat F
+GLIBC_2.41 fwide F
+GLIBC_2.41 fwprintf F
+GLIBC_2.41 fwrite F
+GLIBC_2.41 fwrite_unlocked F
+GLIBC_2.41 fwscanf F
+GLIBC_2.41 gai_cancel F
+GLIBC_2.41 gai_error F
+GLIBC_2.41 gai_strerror F
+GLIBC_2.41 gai_suspend F
+GLIBC_2.41 gcvt F
+GLIBC_2.41 get_avphys_pages F
+GLIBC_2.41 get_current_dir_name F
+GLIBC_2.41 get_nprocs F
+GLIBC_2.41 get_nprocs_conf F
+GLIBC_2.41 get_phys_pages F
+GLIBC_2.41 getaddrinfo F
+GLIBC_2.41 getaddrinfo_a F
+GLIBC_2.41 getaliasbyname F
+GLIBC_2.41 getaliasbyname_r F
+GLIBC_2.41 getaliasent F
+GLIBC_2.41 getaliasent_r F
+GLIBC_2.41 getauxval F
+GLIBC_2.41 getc F
+GLIBC_2.41 getc_unlocked F
+GLIBC_2.41 getchar F
+GLIBC_2.41 getchar_unlocked F
+GLIBC_2.41 getcontext F
+GLIBC_2.41 getcpu F
+GLIBC_2.41 getcwd F
+GLIBC_2.41 getdate F
+GLIBC_2.41 getdate_err D 0x4
+GLIBC_2.41 getdate_r F
+GLIBC_2.41 getdelim F
+GLIBC_2.41 getdents64 F
+GLIBC_2.41 getdirentries F
+GLIBC_2.41 getdirentries64 F
+GLIBC_2.41 getdomainname F
+GLIBC_2.41 getdtablesize F
+GLIBC_2.41 getegid F
+GLIBC_2.41 getentropy F
+GLIBC_2.41 getenv F
+GLIBC_2.41 geteuid F
+GLIBC_2.41 getfsent F
+GLIBC_2.41 getfsfile F
+GLIBC_2.41 getfsspec F
+GLIBC_2.41 getgid F
+GLIBC_2.41 getgrent F
+GLIBC_2.41 getgrent_r F
+GLIBC_2.41 getgrgid F
+GLIBC_2.41 getgrgid_r F
+GLIBC_2.41 getgrnam F
+GLIBC_2.41 getgrnam_r F
+GLIBC_2.41 getgrouplist F
+GLIBC_2.41 getgroups F
+GLIBC_2.41 gethostbyaddr F
+GLIBC_2.41 gethostbyaddr_r F
+GLIBC_2.41 gethostbyname F
+GLIBC_2.41 gethostbyname2 F
+GLIBC_2.41 gethostbyname2_r F
+GLIBC_2.41 gethostbyname_r F
+GLIBC_2.41 gethostent F
+GLIBC_2.41 gethostent_r F
+GLIBC_2.41 gethostid F
+GLIBC_2.41 gethostname F
+GLIBC_2.41 getifaddrs F
+GLIBC_2.41 getipv4sourcefilter F
+GLIBC_2.41 getitimer F
+GLIBC_2.41 getline F
+GLIBC_2.41 getloadavg F
+GLIBC_2.41 getlogin F
+GLIBC_2.41 getlogin_r F
+GLIBC_2.41 getmntent F
+GLIBC_2.41 getmntent_r F
+GLIBC_2.41 getnameinfo F
+GLIBC_2.41 getnetbyaddr F
+GLIBC_2.41 getnetbyaddr_r F
+GLIBC_2.41 getnetbyname F
+GLIBC_2.41 getnetbyname_r F
+GLIBC_2.41 getnetent F
+GLIBC_2.41 getnetent_r F
+GLIBC_2.41 getnetgrent F
+GLIBC_2.41 getnetgrent_r F
+GLIBC_2.41 getopt F
+GLIBC_2.41 getopt_long F
+GLIBC_2.41 getopt_long_only F
+GLIBC_2.41 getpagesize F
+GLIBC_2.41 getpass F
+GLIBC_2.41 getpeername F
+GLIBC_2.41 getpgid F
+GLIBC_2.41 getpgrp F
+GLIBC_2.41 getpid F
+GLIBC_2.41 getppid F
+GLIBC_2.41 getpriority F
+GLIBC_2.41 getprotobyname F
+GLIBC_2.41 getprotobyname_r F
+GLIBC_2.41 getprotobynumber F
+GLIBC_2.41 getprotobynumber_r F
+GLIBC_2.41 getprotoent F
+GLIBC_2.41 getprotoent_r F
+GLIBC_2.41 getpt F
+GLIBC_2.41 getpw F
+GLIBC_2.41 getpwent F
+GLIBC_2.41 getpwent_r F
+GLIBC_2.41 getpwnam F
+GLIBC_2.41 getpwnam_r F
+GLIBC_2.41 getpwuid F
+GLIBC_2.41 getpwuid_r F
+GLIBC_2.41 getrandom F
+GLIBC_2.41 getresgid F
+GLIBC_2.41 getresuid F
+GLIBC_2.41 getrlimit F
+GLIBC_2.41 getrlimit64 F
+GLIBC_2.41 getrpcbyname F
+GLIBC_2.41 getrpcbyname_r F
+GLIBC_2.41 getrpcbynumber F
+GLIBC_2.41 getrpcbynumber_r F
+GLIBC_2.41 getrpcent F
+GLIBC_2.41 getrpcent_r F
+GLIBC_2.41 getrusage F
+GLIBC_2.41 gets F
+GLIBC_2.41 getservbyname F
+GLIBC_2.41 getservbyname_r F
+GLIBC_2.41 getservbyport F
+GLIBC_2.41 getservbyport_r F
+GLIBC_2.41 getservent F
+GLIBC_2.41 getservent_r F
+GLIBC_2.41 getsgent F
+GLIBC_2.41 getsgent_r F
+GLIBC_2.41 getsgnam F
+GLIBC_2.41 getsgnam_r F
+GLIBC_2.41 getsid F
+GLIBC_2.41 getsockname F
+GLIBC_2.41 getsockopt F
+GLIBC_2.41 getsourcefilter F
+GLIBC_2.41 getspent F
+GLIBC_2.41 getspent_r F
+GLIBC_2.41 getspnam F
+GLIBC_2.41 getspnam_r F
+GLIBC_2.41 getsubopt F
+GLIBC_2.41 gettext F
+GLIBC_2.41 gettid F
+GLIBC_2.41 gettimeofday F
+GLIBC_2.41 getttyent F
+GLIBC_2.41 getttynam F
+GLIBC_2.41 getuid F
+GLIBC_2.41 getusershell F
+GLIBC_2.41 getutent F
+GLIBC_2.41 getutent_r F
+GLIBC_2.41 getutid F
+GLIBC_2.41 getutid_r F
+GLIBC_2.41 getutline F
+GLIBC_2.41 getutline_r F
+GLIBC_2.41 getutmp F
+GLIBC_2.41 getutmpx F
+GLIBC_2.41 getutxent F
+GLIBC_2.41 getutxid F
+GLIBC_2.41 getutxline F
+GLIBC_2.41 getw F
+GLIBC_2.41 getwc F
+GLIBC_2.41 getwc_unlocked F
+GLIBC_2.41 getwchar F
+GLIBC_2.41 getwchar_unlocked F
+GLIBC_2.41 getwd F
+GLIBC_2.41 getxattr F
+GLIBC_2.41 glob F
+GLIBC_2.41 glob64 F
+GLIBC_2.41 glob_pattern_p F
+GLIBC_2.41 globfree F
+GLIBC_2.41 globfree64 F
+GLIBC_2.41 gmtime F
+GLIBC_2.41 gmtime_r F
+GLIBC_2.41 gnu_dev_major F
+GLIBC_2.41 gnu_dev_makedev F
+GLIBC_2.41 gnu_dev_minor F
+GLIBC_2.41 gnu_get_libc_release F
+GLIBC_2.41 gnu_get_libc_version F
+GLIBC_2.41 grantpt F
+GLIBC_2.41 group_member F
+GLIBC_2.41 gsignal F
+GLIBC_2.41 gtty F
+GLIBC_2.41 h_errlist D 0x14
+GLIBC_2.41 h_nerr D 0x4
+GLIBC_2.41 hasmntopt F
+GLIBC_2.41 hcreate F
+GLIBC_2.41 hcreate_r F
+GLIBC_2.41 hdestroy F
+GLIBC_2.41 hdestroy_r F
+GLIBC_2.41 herror F
+GLIBC_2.41 hsearch F
+GLIBC_2.41 hsearch_r F
+GLIBC_2.41 hstrerror F
+GLIBC_2.41 htonl F
+GLIBC_2.41 htons F
+GLIBC_2.41 iconv F
+GLIBC_2.41 iconv_close F
+GLIBC_2.41 iconv_open F
+GLIBC_2.41 if_freenameindex F
+GLIBC_2.41 if_indextoname F
+GLIBC_2.41 if_nameindex F
+GLIBC_2.41 if_nametoindex F
+GLIBC_2.41 imaxabs F
+GLIBC_2.41 imaxdiv F
+GLIBC_2.41 in6addr_any D 0x10
+GLIBC_2.41 in6addr_loopback D 0x10
+GLIBC_2.41 index F
+GLIBC_2.41 inet6_opt_append F
+GLIBC_2.41 inet6_opt_find F
+GLIBC_2.41 inet6_opt_finish F
+GLIBC_2.41 inet6_opt_get_val F
+GLIBC_2.41 inet6_opt_init F
+GLIBC_2.41 inet6_opt_next F
+GLIBC_2.41 inet6_opt_set_val F
+GLIBC_2.41 inet6_option_alloc F
+GLIBC_2.41 inet6_option_append F
+GLIBC_2.41 inet6_option_find F
+GLIBC_2.41 inet6_option_init F
+GLIBC_2.41 inet6_option_next F
+GLIBC_2.41 inet6_option_space F
+GLIBC_2.41 inet6_rth_add F
+GLIBC_2.41 inet6_rth_getaddr F
+GLIBC_2.41 inet6_rth_init F
+GLIBC_2.41 inet6_rth_reverse F
+GLIBC_2.41 inet6_rth_segments F
+GLIBC_2.41 inet6_rth_space F
+GLIBC_2.41 inet_addr F
+GLIBC_2.41 inet_aton F
+GLIBC_2.41 inet_lnaof F
+GLIBC_2.41 inet_makeaddr F
+GLIBC_2.41 inet_netof F
+GLIBC_2.41 inet_network F
+GLIBC_2.41 inet_nsap_addr F
+GLIBC_2.41 inet_nsap_ntoa F
+GLIBC_2.41 inet_ntoa F
+GLIBC_2.41 inet_ntop F
+GLIBC_2.41 inet_pton F
+GLIBC_2.41 init_module F
+GLIBC_2.41 initgroups F
+GLIBC_2.41 initstate F
+GLIBC_2.41 initstate_r F
+GLIBC_2.41 innetgr F
+GLIBC_2.41 inotify_add_watch F
+GLIBC_2.41 inotify_init F
+GLIBC_2.41 inotify_init1 F
+GLIBC_2.41 inotify_rm_watch F
+GLIBC_2.41 insque F
+GLIBC_2.41 ioctl F
+GLIBC_2.41 iruserok F
+GLIBC_2.41 iruserok_af F
+GLIBC_2.41 isalnum F
+GLIBC_2.41 isalnum_l F
+GLIBC_2.41 isalpha F
+GLIBC_2.41 isalpha_l F
+GLIBC_2.41 isascii F
+GLIBC_2.41 isatty F
+GLIBC_2.41 isblank F
+GLIBC_2.41 isblank_l F
+GLIBC_2.41 iscntrl F
+GLIBC_2.41 iscntrl_l F
+GLIBC_2.41 isctype F
+GLIBC_2.41 isdigit F
+GLIBC_2.41 isdigit_l F
+GLIBC_2.41 isfdtype F
+GLIBC_2.41 isgraph F
+GLIBC_2.41 isgraph_l F
+GLIBC_2.41 isinf F
+GLIBC_2.41 isinff F
+GLIBC_2.41 isinfl F
+GLIBC_2.41 islower F
+GLIBC_2.41 islower_l F
+GLIBC_2.41 isnan F
+GLIBC_2.41 isnanf F
+GLIBC_2.41 isnanl F
+GLIBC_2.41 isprint F
+GLIBC_2.41 isprint_l F
+GLIBC_2.41 ispunct F
+GLIBC_2.41 ispunct_l F
+GLIBC_2.41 isspace F
+GLIBC_2.41 isspace_l F
+GLIBC_2.41 isupper F
+GLIBC_2.41 isupper_l F
+GLIBC_2.41 iswalnum F
+GLIBC_2.41 iswalnum_l F
+GLIBC_2.41 iswalpha F
+GLIBC_2.41 iswalpha_l F
+GLIBC_2.41 iswblank F
+GLIBC_2.41 iswblank_l F
+GLIBC_2.41 iswcntrl F
+GLIBC_2.41 iswcntrl_l F
+GLIBC_2.41 iswctype F
+GLIBC_2.41 iswctype_l F
+GLIBC_2.41 iswdigit F
+GLIBC_2.41 iswdigit_l F
+GLIBC_2.41 iswgraph F
+GLIBC_2.41 iswgraph_l F
+GLIBC_2.41 iswlower F
+GLIBC_2.41 iswlower_l F
+GLIBC_2.41 iswprint F
+GLIBC_2.41 iswprint_l F
+GLIBC_2.41 iswpunct F
+GLIBC_2.41 iswpunct_l F
+GLIBC_2.41 iswspace F
+GLIBC_2.41 iswspace_l F
+GLIBC_2.41 iswupper F
+GLIBC_2.41 iswupper_l F
+GLIBC_2.41 iswxdigit F
+GLIBC_2.41 iswxdigit_l F
+GLIBC_2.41 isxdigit F
+GLIBC_2.41 isxdigit_l F
+GLIBC_2.41 jrand48 F
+GLIBC_2.41 jrand48_r F
+GLIBC_2.41 kill F
+GLIBC_2.41 killpg F
+GLIBC_2.41 klogctl F
+GLIBC_2.41 l64a F
+GLIBC_2.41 labs F
+GLIBC_2.41 lchmod F
+GLIBC_2.41 lchown F
+GLIBC_2.41 lckpwdf F
+GLIBC_2.41 lcong48 F
+GLIBC_2.41 lcong48_r F
+GLIBC_2.41 ldexp F
+GLIBC_2.41 ldexpf F
+GLIBC_2.41 ldexpl F
+GLIBC_2.41 ldiv F
+GLIBC_2.41 lfind F
+GLIBC_2.41 lgetxattr F
+GLIBC_2.41 link F
+GLIBC_2.41 linkat F
+GLIBC_2.41 lio_listio F
+GLIBC_2.41 lio_listio64 F
+GLIBC_2.41 listen F
+GLIBC_2.41 listxattr F
+GLIBC_2.41 llabs F
+GLIBC_2.41 lldiv F
+GLIBC_2.41 llistxattr F
+GLIBC_2.41 localeconv F
+GLIBC_2.41 localtime F
+GLIBC_2.41 localtime_r F
+GLIBC_2.41 lockf F
+GLIBC_2.41 lockf64 F
+GLIBC_2.41 login F
+GLIBC_2.41 login_tty F
+GLIBC_2.41 logout F
+GLIBC_2.41 logwtmp F
+GLIBC_2.41 longjmp F
+GLIBC_2.41 lrand48 F
+GLIBC_2.41 lrand48_r F
+GLIBC_2.41 lremovexattr F
+GLIBC_2.41 lsearch F
+GLIBC_2.41 lseek F
+GLIBC_2.41 lseek64 F
+GLIBC_2.41 lsetxattr F
+GLIBC_2.41 lstat F
+GLIBC_2.41 lstat64 F
+GLIBC_2.41 lutimes F
+GLIBC_2.41 madvise F
+GLIBC_2.41 makecontext F
+GLIBC_2.41 mallinfo F
+GLIBC_2.41 mallinfo2 F
+GLIBC_2.41 malloc F
+GLIBC_2.41 malloc_info F
+GLIBC_2.41 malloc_stats F
+GLIBC_2.41 malloc_trim F
+GLIBC_2.41 malloc_usable_size F
+GLIBC_2.41 mallopt F
+GLIBC_2.41 mblen F
+GLIBC_2.41 mbrlen F
+GLIBC_2.41 mbrtoc16 F
+GLIBC_2.41 mbrtoc32 F
+GLIBC_2.41 mbrtoc8 F
+GLIBC_2.41 mbrtowc F
+GLIBC_2.41 mbsinit F
+GLIBC_2.41 mbsnrtowcs F
+GLIBC_2.41 mbsrtowcs F
+GLIBC_2.41 mbstowcs F
+GLIBC_2.41 mbtowc F
+GLIBC_2.41 mcheck F
+GLIBC_2.41 mcheck_check_all F
+GLIBC_2.41 mcheck_pedantic F
+GLIBC_2.41 memalign F
+GLIBC_2.41 memccpy F
+GLIBC_2.41 memchr F
+GLIBC_2.41 memcmp F
+GLIBC_2.41 memcpy F
+GLIBC_2.41 memfd_create F
+GLIBC_2.41 memfrob F
+GLIBC_2.41 memmem F
+GLIBC_2.41 memmove F
+GLIBC_2.41 mempcpy F
+GLIBC_2.41 memrchr F
+GLIBC_2.41 memset F
+GLIBC_2.41 mincore F
+GLIBC_2.41 mkdir F
+GLIBC_2.41 mkdirat F
+GLIBC_2.41 mkdtemp F
+GLIBC_2.41 mkfifo F
+GLIBC_2.41 mkfifoat F
+GLIBC_2.41 mknod F
+GLIBC_2.41 mknodat F
+GLIBC_2.41 mkostemp F
+GLIBC_2.41 mkostemp64 F
+GLIBC_2.41 mkostemps F
+GLIBC_2.41 mkostemps64 F
+GLIBC_2.41 mkstemp F
+GLIBC_2.41 mkstemp64 F
+GLIBC_2.41 mkstemps F
+GLIBC_2.41 mkstemps64 F
+GLIBC_2.41 mktemp F
+GLIBC_2.41 mktime F
+GLIBC_2.41 mlock F
+GLIBC_2.41 mlock2 F
+GLIBC_2.41 mlockall F
+GLIBC_2.41 mmap F
+GLIBC_2.41 mmap64 F
+GLIBC_2.41 modf F
+GLIBC_2.41 modff F
+GLIBC_2.41 modfl F
+GLIBC_2.41 moncontrol F
+GLIBC_2.41 monstartup F
+GLIBC_2.41 mount F
+GLIBC_2.41 mount_setattr F
+GLIBC_2.41 move_mount F
+GLIBC_2.41 mprobe F
+GLIBC_2.41 mprotect F
+GLIBC_2.41 mq_close F
+GLIBC_2.41 mq_getattr F
+GLIBC_2.41 mq_notify F
+GLIBC_2.41 mq_open F
+GLIBC_2.41 mq_receive F
+GLIBC_2.41 mq_send F
+GLIBC_2.41 mq_setattr F
+GLIBC_2.41 mq_timedreceive F
+GLIBC_2.41 mq_timedsend F
+GLIBC_2.41 mq_unlink F
+GLIBC_2.41 mrand48 F
+GLIBC_2.41 mrand48_r F
+GLIBC_2.41 mremap F
+GLIBC_2.41 msgctl F
+GLIBC_2.41 msgget F
+GLIBC_2.41 msgrcv F
+GLIBC_2.41 msgsnd F
+GLIBC_2.41 msync F
+GLIBC_2.41 mtrace F
+GLIBC_2.41 mtx_destroy F
+GLIBC_2.41 mtx_init F
+GLIBC_2.41 mtx_lock F
+GLIBC_2.41 mtx_timedlock F
+GLIBC_2.41 mtx_trylock F
+GLIBC_2.41 mtx_unlock F
+GLIBC_2.41 munlock F
+GLIBC_2.41 munlockall F
+GLIBC_2.41 munmap F
+GLIBC_2.41 muntrace F
+GLIBC_2.41 name_to_handle_at F
+GLIBC_2.41 nanosleep F
+GLIBC_2.41 newlocale F
+GLIBC_2.41 nftw F
+GLIBC_2.41 nftw64 F
+GLIBC_2.41 ngettext F
+GLIBC_2.41 nice F
+GLIBC_2.41 nl_langinfo F
+GLIBC_2.41 nl_langinfo_l F
+GLIBC_2.41 nrand48 F
+GLIBC_2.41 nrand48_r F
+GLIBC_2.41 ns_name_compress F
+GLIBC_2.41 ns_name_ntop F
+GLIBC_2.41 ns_name_pack F
+GLIBC_2.41 ns_name_pton F
+GLIBC_2.41 ns_name_skip F
+GLIBC_2.41 ns_name_uncompress F
+GLIBC_2.41 ns_name_unpack F
+GLIBC_2.41 ntohl F
+GLIBC_2.41 ntohs F
+GLIBC_2.41 ntp_adjtime F
+GLIBC_2.41 ntp_gettime F
+GLIBC_2.41 ntp_gettimex F
+GLIBC_2.41 obstack_alloc_failed_handler D 0x4
+GLIBC_2.41 obstack_exit_failure D 0x4
+GLIBC_2.41 obstack_free F
+GLIBC_2.41 obstack_printf F
+GLIBC_2.41 obstack_vprintf F
+GLIBC_2.41 on_exit F
+GLIBC_2.41 open F
+GLIBC_2.41 open64 F
+GLIBC_2.41 open_by_handle_at F
+GLIBC_2.41 open_memstream F
+GLIBC_2.41 open_tree F
+GLIBC_2.41 open_wmemstream F
+GLIBC_2.41 openat F
+GLIBC_2.41 openat64 F
+GLIBC_2.41 opendir F
+GLIBC_2.41 openlog F
+GLIBC_2.41 openpty F
+GLIBC_2.41 optarg D 0x4
+GLIBC_2.41 opterr D 0x4
+GLIBC_2.41 optind D 0x4
+GLIBC_2.41 optopt D 0x4
+GLIBC_2.41 parse_printf_format F
+GLIBC_2.41 pathconf F
+GLIBC_2.41 pause F
+GLIBC_2.41 pclose F
+GLIBC_2.41 perror F
+GLIBC_2.41 personality F
+GLIBC_2.41 pidfd_getfd F
+GLIBC_2.41 pidfd_getpid F
+GLIBC_2.41 pidfd_open F
+GLIBC_2.41 pidfd_send_signal F
+GLIBC_2.41 pidfd_spawn F
+GLIBC_2.41 pidfd_spawnp F
+GLIBC_2.41 pipe F
+GLIBC_2.41 pipe2 F
+GLIBC_2.41 pivot_root F
+GLIBC_2.41 pkey_alloc F
+GLIBC_2.41 pkey_free F
+GLIBC_2.41 pkey_get F
+GLIBC_2.41 pkey_mprotect F
+GLIBC_2.41 pkey_set F
+GLIBC_2.41 poll F
+GLIBC_2.41 popen F
+GLIBC_2.41 posix_fadvise F
+GLIBC_2.41 posix_fadvise64 F
+GLIBC_2.41 posix_fallocate F
+GLIBC_2.41 posix_fallocate64 F
+GLIBC_2.41 posix_madvise F
+GLIBC_2.41 posix_memalign F
+GLIBC_2.41 posix_openpt F
+GLIBC_2.41 posix_spawn F
+GLIBC_2.41 posix_spawn_file_actions_addchdir_np F
+GLIBC_2.41 posix_spawn_file_actions_addclose F
+GLIBC_2.41 posix_spawn_file_actions_addclosefrom_np F
+GLIBC_2.41 posix_spawn_file_actions_adddup2 F
+GLIBC_2.41 posix_spawn_file_actions_addfchdir_np F
+GLIBC_2.41 posix_spawn_file_actions_addopen F
+GLIBC_2.41 posix_spawn_file_actions_addtcsetpgrp_np F
+GLIBC_2.41 posix_spawn_file_actions_destroy F
+GLIBC_2.41 posix_spawn_file_actions_init F
+GLIBC_2.41 posix_spawnattr_destroy F
+GLIBC_2.41 posix_spawnattr_getcgroup_np F
+GLIBC_2.41 posix_spawnattr_getflags F
+GLIBC_2.41 posix_spawnattr_getpgroup F
+GLIBC_2.41 posix_spawnattr_getschedparam F
+GLIBC_2.41 posix_spawnattr_getschedpolicy F
+GLIBC_2.41 posix_spawnattr_getsigdefault F
+GLIBC_2.41 posix_spawnattr_getsigmask F
+GLIBC_2.41 posix_spawnattr_init F
+GLIBC_2.41 posix_spawnattr_setcgroup_np F
+GLIBC_2.41 posix_spawnattr_setflags F
+GLIBC_2.41 posix_spawnattr_setpgroup F
+GLIBC_2.41 posix_spawnattr_setschedparam F
+GLIBC_2.41 posix_spawnattr_setschedpolicy F
+GLIBC_2.41 posix_spawnattr_setsigdefault F
+GLIBC_2.41 posix_spawnattr_setsigmask F
+GLIBC_2.41 posix_spawnp F
+GLIBC_2.41 ppoll F
+GLIBC_2.41 prctl F
+GLIBC_2.41 pread F
+GLIBC_2.41 pread64 F
+GLIBC_2.41 preadv F
+GLIBC_2.41 preadv2 F
+GLIBC_2.41 preadv64 F
+GLIBC_2.41 preadv64v2 F
+GLIBC_2.41 printf F
+GLIBC_2.41 printf_size F
+GLIBC_2.41 printf_size_info F
+GLIBC_2.41 prlimit F
+GLIBC_2.41 prlimit64 F
+GLIBC_2.41 process_madvise F
+GLIBC_2.41 process_mrelease F
+GLIBC_2.41 process_vm_readv F
+GLIBC_2.41 process_vm_writev F
+GLIBC_2.41 profil F
+GLIBC_2.41 program_invocation_name D 0x4
+GLIBC_2.41 program_invocation_short_name D 0x4
+GLIBC_2.41 pselect F
+GLIBC_2.41 psiginfo F
+GLIBC_2.41 psignal F
+GLIBC_2.41 pthread_attr_destroy F
+GLIBC_2.41 pthread_attr_getaffinity_np F
+GLIBC_2.41 pthread_attr_getdetachstate F
+GLIBC_2.41 pthread_attr_getguardsize F
+GLIBC_2.41 pthread_attr_getinheritsched F
+GLIBC_2.41 pthread_attr_getschedparam F
+GLIBC_2.41 pthread_attr_getschedpolicy F
+GLIBC_2.41 pthread_attr_getscope F
+GLIBC_2.41 pthread_attr_getsigmask_np F
+GLIBC_2.41 pthread_attr_getstack F
+GLIBC_2.41 pthread_attr_getstackaddr F
+GLIBC_2.41 pthread_attr_getstacksize F
+GLIBC_2.41 pthread_attr_init F
+GLIBC_2.41 pthread_attr_setaffinity_np F
+GLIBC_2.41 pthread_attr_setdetachstate F
+GLIBC_2.41 pthread_attr_setguardsize F
+GLIBC_2.41 pthread_attr_setinheritsched F
+GLIBC_2.41 pthread_attr_setschedparam F
+GLIBC_2.41 pthread_attr_setschedpolicy F
+GLIBC_2.41 pthread_attr_setscope F
+GLIBC_2.41 pthread_attr_setsigmask_np F
+GLIBC_2.41 pthread_attr_setstack F
+GLIBC_2.41 pthread_attr_setstackaddr F
+GLIBC_2.41 pthread_attr_setstacksize F
+GLIBC_2.41 pthread_barrier_destroy F
+GLIBC_2.41 pthread_barrier_init F
+GLIBC_2.41 pthread_barrier_wait F
+GLIBC_2.41 pthread_barrierattr_destroy F
+GLIBC_2.41 pthread_barrierattr_getpshared F
+GLIBC_2.41 pthread_barrierattr_init F
+GLIBC_2.41 pthread_barrierattr_setpshared F
+GLIBC_2.41 pthread_cancel F
+GLIBC_2.41 pthread_clockjoin_np F
+GLIBC_2.41 pthread_cond_broadcast F
+GLIBC_2.41 pthread_cond_clockwait F
+GLIBC_2.41 pthread_cond_destroy F
+GLIBC_2.41 pthread_cond_init F
+GLIBC_2.41 pthread_cond_signal F
+GLIBC_2.41 pthread_cond_timedwait F
+GLIBC_2.41 pthread_cond_wait F
+GLIBC_2.41 pthread_condattr_destroy F
+GLIBC_2.41 pthread_condattr_getclock F
+GLIBC_2.41 pthread_condattr_getpshared F
+GLIBC_2.41 pthread_condattr_init F
+GLIBC_2.41 pthread_condattr_setclock F
+GLIBC_2.41 pthread_condattr_setpshared F
+GLIBC_2.41 pthread_create F
+GLIBC_2.41 pthread_detach F
+GLIBC_2.41 pthread_equal F
+GLIBC_2.41 pthread_exit F
+GLIBC_2.41 pthread_getaffinity_np F
+GLIBC_2.41 pthread_getattr_default_np F
+GLIBC_2.41 pthread_getattr_np F
+GLIBC_2.41 pthread_getconcurrency F
+GLIBC_2.41 pthread_getcpuclockid F
+GLIBC_2.41 pthread_getname_np F
+GLIBC_2.41 pthread_getschedparam F
+GLIBC_2.41 pthread_getspecific F
+GLIBC_2.41 pthread_join F
+GLIBC_2.41 pthread_key_create F
+GLIBC_2.41 pthread_key_delete F
+GLIBC_2.41 pthread_kill F
+GLIBC_2.41 pthread_mutex_clocklock F
+GLIBC_2.41 pthread_mutex_consistent F
+GLIBC_2.41 pthread_mutex_destroy F
+GLIBC_2.41 pthread_mutex_getprioceiling F
+GLIBC_2.41 pthread_mutex_init F
+GLIBC_2.41 pthread_mutex_lock F
+GLIBC_2.41 pthread_mutex_setprioceiling F
+GLIBC_2.41 pthread_mutex_timedlock F
+GLIBC_2.41 pthread_mutex_trylock F
+GLIBC_2.41 pthread_mutex_unlock F
+GLIBC_2.41 pthread_mutexattr_destroy F
+GLIBC_2.41 pthread_mutexattr_getprioceiling F
+GLIBC_2.41 pthread_mutexattr_getprotocol F
+GLIBC_2.41 pthread_mutexattr_getpshared F
+GLIBC_2.41 pthread_mutexattr_getrobust F
+GLIBC_2.41 pthread_mutexattr_gettype F
+GLIBC_2.41 pthread_mutexattr_init F
+GLIBC_2.41 pthread_mutexattr_setprioceiling F
+GLIBC_2.41 pthread_mutexattr_setprotocol F
+GLIBC_2.41 pthread_mutexattr_setpshared F
+GLIBC_2.41 pthread_mutexattr_setrobust F
+GLIBC_2.41 pthread_mutexattr_settype F
+GLIBC_2.41 pthread_once F
+GLIBC_2.41 pthread_rwlock_clockrdlock F
+GLIBC_2.41 pthread_rwlock_clockwrlock F
+GLIBC_2.41 pthread_rwlock_destroy F
+GLIBC_2.41 pthread_rwlock_init F
+GLIBC_2.41 pthread_rwlock_rdlock F
+GLIBC_2.41 pthread_rwlock_timedrdlock F
+GLIBC_2.41 pthread_rwlock_timedwrlock F
+GLIBC_2.41 pthread_rwlock_tryrdlock F
+GLIBC_2.41 pthread_rwlock_trywrlock F
+GLIBC_2.41 pthread_rwlock_unlock F
+GLIBC_2.41 pthread_rwlock_wrlock F
+GLIBC_2.41 pthread_rwlockattr_destroy F
+GLIBC_2.41 pthread_rwlockattr_getkind_np F
+GLIBC_2.41 pthread_rwlockattr_getpshared F
+GLIBC_2.41 pthread_rwlockattr_init F
+GLIBC_2.41 pthread_rwlockattr_setkind_np F
+GLIBC_2.41 pthread_rwlockattr_setpshared F
+GLIBC_2.41 pthread_self F
+GLIBC_2.41 pthread_setaffinity_np F
+GLIBC_2.41 pthread_setattr_default_np F
+GLIBC_2.41 pthread_setcancelstate F
+GLIBC_2.41 pthread_setcanceltype F
+GLIBC_2.41 pthread_setconcurrency F
+GLIBC_2.41 pthread_setname_np F
+GLIBC_2.41 pthread_setschedparam F
+GLIBC_2.41 pthread_setschedprio F
+GLIBC_2.41 pthread_setspecific F
+GLIBC_2.41 pthread_sigmask F
+GLIBC_2.41 pthread_sigqueue F
+GLIBC_2.41 pthread_spin_destroy F
+GLIBC_2.41 pthread_spin_init F
+GLIBC_2.41 pthread_spin_lock F
+GLIBC_2.41 pthread_spin_trylock F
+GLIBC_2.41 pthread_spin_unlock F
+GLIBC_2.41 pthread_testcancel F
+GLIBC_2.41 pthread_timedjoin_np F
+GLIBC_2.41 pthread_tryjoin_np F
+GLIBC_2.41 ptrace F
+GLIBC_2.41 ptsname F
+GLIBC_2.41 ptsname_r F
+GLIBC_2.41 putc F
+GLIBC_2.41 putc_unlocked F
+GLIBC_2.41 putchar F
+GLIBC_2.41 putchar_unlocked F
+GLIBC_2.41 putenv F
+GLIBC_2.41 putgrent F
+GLIBC_2.41 putpwent F
+GLIBC_2.41 puts F
+GLIBC_2.41 putsgent F
+GLIBC_2.41 putspent F
+GLIBC_2.41 pututline F
+GLIBC_2.41 pututxline F
+GLIBC_2.41 putw F
+GLIBC_2.41 putwc F
+GLIBC_2.41 putwc_unlocked F
+GLIBC_2.41 putwchar F
+GLIBC_2.41 putwchar_unlocked F
+GLIBC_2.41 pvalloc F
+GLIBC_2.41 pwrite F
+GLIBC_2.41 pwrite64 F
+GLIBC_2.41 pwritev F
+GLIBC_2.41 pwritev2 F
+GLIBC_2.41 pwritev64 F
+GLIBC_2.41 pwritev64v2 F
+GLIBC_2.41 qecvt F
+GLIBC_2.41 qecvt_r F
+GLIBC_2.41 qfcvt F
+GLIBC_2.41 qfcvt_r F
+GLIBC_2.41 qgcvt F
+GLIBC_2.41 qsort F
+GLIBC_2.41 qsort_r F
+GLIBC_2.41 quick_exit F
+GLIBC_2.41 quotactl F
+GLIBC_2.41 raise F
+GLIBC_2.41 rand F
+GLIBC_2.41 rand_r F
+GLIBC_2.41 random F
+GLIBC_2.41 random_r F
+GLIBC_2.41 rawmemchr F
+GLIBC_2.41 rcmd F
+GLIBC_2.41 rcmd_af F
+GLIBC_2.41 re_comp F
+GLIBC_2.41 re_compile_fastmap F
+GLIBC_2.41 re_compile_pattern F
+GLIBC_2.41 re_exec F
+GLIBC_2.41 re_match F
+GLIBC_2.41 re_match_2 F
+GLIBC_2.41 re_search F
+GLIBC_2.41 re_search_2 F
+GLIBC_2.41 re_set_registers F
+GLIBC_2.41 re_set_syntax F
+GLIBC_2.41 re_syntax_options D 0x4
+GLIBC_2.41 read F
+GLIBC_2.41 readahead F
+GLIBC_2.41 readdir F
+GLIBC_2.41 readdir64 F
+GLIBC_2.41 readdir64_r F
+GLIBC_2.41 readdir_r F
+GLIBC_2.41 readlink F
+GLIBC_2.41 readlinkat F
+GLIBC_2.41 readv F
+GLIBC_2.41 realloc F
+GLIBC_2.41 reallocarray F
+GLIBC_2.41 realpath F
+GLIBC_2.41 reboot F
+GLIBC_2.41 recv F
+GLIBC_2.41 recvfrom F
+GLIBC_2.41 recvmmsg F
+GLIBC_2.41 recvmsg F
+GLIBC_2.41 regcomp F
+GLIBC_2.41 regerror F
+GLIBC_2.41 regexec F
+GLIBC_2.41 regfree F
+GLIBC_2.41 register_printf_function F
+GLIBC_2.41 register_printf_modifier F
+GLIBC_2.41 register_printf_specifier F
+GLIBC_2.41 register_printf_type F
+GLIBC_2.41 remap_file_pages F
+GLIBC_2.41 remove F
+GLIBC_2.41 removexattr F
+GLIBC_2.41 remque F
+GLIBC_2.41 rename F
+GLIBC_2.41 renameat F
+GLIBC_2.41 renameat2 F
+GLIBC_2.41 res_dnok F
+GLIBC_2.41 res_hnok F
+GLIBC_2.41 res_mailok F
+GLIBC_2.41 res_mkquery F
+GLIBC_2.41 res_nmkquery F
+GLIBC_2.41 res_nquery F
+GLIBC_2.41 res_nquerydomain F
+GLIBC_2.41 res_nsearch F
+GLIBC_2.41 res_nsend F
+GLIBC_2.41 res_ownok F
+GLIBC_2.41 res_query F
+GLIBC_2.41 res_querydomain F
+GLIBC_2.41 res_search F
+GLIBC_2.41 res_send F
+GLIBC_2.41 revoke F
+GLIBC_2.41 rewind F
+GLIBC_2.41 rewinddir F
+GLIBC_2.41 rexec F
+GLIBC_2.41 rexec_af F
+GLIBC_2.41 rexecoptions D 0x4
+GLIBC_2.41 rindex F
+GLIBC_2.41 rmdir F
+GLIBC_2.41 rpmatch F
+GLIBC_2.41 rresvport F
+GLIBC_2.41 rresvport_af F
+GLIBC_2.41 ruserok F
+GLIBC_2.41 ruserok_af F
+GLIBC_2.41 ruserpass F
+GLIBC_2.41 sbrk F
+GLIBC_2.41 scalbn F
+GLIBC_2.41 scalbnf F
+GLIBC_2.41 scalbnl F
+GLIBC_2.41 scandir F
+GLIBC_2.41 scandir64 F
+GLIBC_2.41 scandirat F
+GLIBC_2.41 scandirat64 F
+GLIBC_2.41 scanf F
+GLIBC_2.41 sched_get_priority_max F
+GLIBC_2.41 sched_get_priority_min F
+GLIBC_2.41 sched_getaffinity F
+GLIBC_2.41 sched_getattr F
+GLIBC_2.41 sched_getcpu F
+GLIBC_2.41 sched_getparam F
+GLIBC_2.41 sched_getscheduler F
+GLIBC_2.41 sched_rr_get_interval F
+GLIBC_2.41 sched_setaffinity F
+GLIBC_2.41 sched_setattr F
+GLIBC_2.41 sched_setparam F
+GLIBC_2.41 sched_setscheduler F
+GLIBC_2.41 sched_yield F
+GLIBC_2.41 secure_getenv F
+GLIBC_2.41 seed48 F
+GLIBC_2.41 seed48_r F
+GLIBC_2.41 seekdir F
+GLIBC_2.41 select F
+GLIBC_2.41 sem_clockwait F
+GLIBC_2.41 sem_close F
+GLIBC_2.41 sem_destroy F
+GLIBC_2.41 sem_getvalue F
+GLIBC_2.41 sem_init F
+GLIBC_2.41 sem_open F
+GLIBC_2.41 sem_post F
+GLIBC_2.41 sem_timedwait F
+GLIBC_2.41 sem_trywait F
+GLIBC_2.41 sem_unlink F
+GLIBC_2.41 sem_wait F
+GLIBC_2.41 semctl F
+GLIBC_2.41 semget F
+GLIBC_2.41 semop F
+GLIBC_2.41 semtimedop F
+GLIBC_2.41 send F
+GLIBC_2.41 sendfile F
+GLIBC_2.41 sendfile64 F
+GLIBC_2.41 sendmmsg F
+GLIBC_2.41 sendmsg F
+GLIBC_2.41 sendto F
+GLIBC_2.41 setaliasent F
+GLIBC_2.41 setbuf F
+GLIBC_2.41 setbuffer F
+GLIBC_2.41 setcontext F
+GLIBC_2.41 setdomainname F
+GLIBC_2.41 setegid F
+GLIBC_2.41 setenv F
+GLIBC_2.41 seteuid F
+GLIBC_2.41 setfsent F
+GLIBC_2.41 setfsgid F
+GLIBC_2.41 setfsuid F
+GLIBC_2.41 setgid F
+GLIBC_2.41 setgrent F
+GLIBC_2.41 setgroups F
+GLIBC_2.41 sethostent F
+GLIBC_2.41 sethostid F
+GLIBC_2.41 sethostname F
+GLIBC_2.41 setipv4sourcefilter F
+GLIBC_2.41 setitimer F
+GLIBC_2.41 setjmp F
+GLIBC_2.41 setlinebuf F
+GLIBC_2.41 setlocale F
+GLIBC_2.41 setlogin F
+GLIBC_2.41 setlogmask F
+GLIBC_2.41 setmntent F
+GLIBC_2.41 setnetent F
+GLIBC_2.41 setnetgrent F
+GLIBC_2.41 setns F
+GLIBC_2.41 setpgid F
+GLIBC_2.41 setpgrp F
+GLIBC_2.41 setpriority F
+GLIBC_2.41 setprotoent F
+GLIBC_2.41 setpwent F
+GLIBC_2.41 setregid F
+GLIBC_2.41 setresgid F
+GLIBC_2.41 setresuid F
+GLIBC_2.41 setreuid F
+GLIBC_2.41 setrlimit F
+GLIBC_2.41 setrlimit64 F
+GLIBC_2.41 setrpcent F
+GLIBC_2.41 setservent F
+GLIBC_2.41 setsgent F
+GLIBC_2.41 setsid F
+GLIBC_2.41 setsockopt F
+GLIBC_2.41 setsourcefilter F
+GLIBC_2.41 setspent F
+GLIBC_2.41 setstate F
+GLIBC_2.41 setstate_r F
+GLIBC_2.41 settimeofday F
+GLIBC_2.41 setttyent F
+GLIBC_2.41 setuid F
+GLIBC_2.41 setusershell F
+GLIBC_2.41 setutent F
+GLIBC_2.41 setutxent F
+GLIBC_2.41 setvbuf F
+GLIBC_2.41 setxattr F
+GLIBC_2.41 sgetsgent F
+GLIBC_2.41 sgetsgent_r F
+GLIBC_2.41 sgetspent F
+GLIBC_2.41 sgetspent_r F
+GLIBC_2.41 shm_open F
+GLIBC_2.41 shm_unlink F
+GLIBC_2.41 shmat F
+GLIBC_2.41 shmctl F
+GLIBC_2.41 shmdt F
+GLIBC_2.41 shmget F
+GLIBC_2.41 shutdown F
+GLIBC_2.41 sigabbrev_np F
+GLIBC_2.41 sigaction F
+GLIBC_2.41 sigaddset F
+GLIBC_2.41 sigaltstack F
+GLIBC_2.41 sigandset F
+GLIBC_2.41 sigblock F
+GLIBC_2.41 sigdelset F
+GLIBC_2.41 sigdescr_np F
+GLIBC_2.41 sigemptyset F
+GLIBC_2.41 sigfillset F
+GLIBC_2.41 siggetmask F
+GLIBC_2.41 sighold F
+GLIBC_2.41 sigignore F
+GLIBC_2.41 siginterrupt F
+GLIBC_2.41 sigisemptyset F
+GLIBC_2.41 sigismember F
+GLIBC_2.41 siglongjmp F
+GLIBC_2.41 signal F
+GLIBC_2.41 signalfd F
+GLIBC_2.41 sigorset F
+GLIBC_2.41 sigpause F
+GLIBC_2.41 sigpending F
+GLIBC_2.41 sigprocmask F
+GLIBC_2.41 sigqueue F
+GLIBC_2.41 sigrelse F
+GLIBC_2.41 sigreturn F
+GLIBC_2.41 sigset F
+GLIBC_2.41 sigsetmask F
+GLIBC_2.41 sigstack F
+GLIBC_2.41 sigsuspend F
+GLIBC_2.41 sigtimedwait F
+GLIBC_2.41 sigwait F
+GLIBC_2.41 sigwaitinfo F
+GLIBC_2.41 sleep F
+GLIBC_2.41 snprintf F
+GLIBC_2.41 sockatmark F
+GLIBC_2.41 socket F
+GLIBC_2.41 socketpair F
+GLIBC_2.41 splice F
+GLIBC_2.41 sprintf F
+GLIBC_2.41 sprofil F
+GLIBC_2.41 srand F
+GLIBC_2.41 srand48 F
+GLIBC_2.41 srand48_r F
+GLIBC_2.41 srandom F
+GLIBC_2.41 srandom_r F
+GLIBC_2.41 sscanf F
+GLIBC_2.41 ssignal F
+GLIBC_2.41 stat F
+GLIBC_2.41 stat64 F
+GLIBC_2.41 statfs F
+GLIBC_2.41 statfs64 F
+GLIBC_2.41 statvfs F
+GLIBC_2.41 statvfs64 F
+GLIBC_2.41 statx F
+GLIBC_2.41 stdc_bit_ceil_uc F
+GLIBC_2.41 stdc_bit_ceil_ui F
+GLIBC_2.41 stdc_bit_ceil_ul F
+GLIBC_2.41 stdc_bit_ceil_ull F
+GLIBC_2.41 stdc_bit_ceil_us F
+GLIBC_2.41 stdc_bit_floor_uc F
+GLIBC_2.41 stdc_bit_floor_ui F
+GLIBC_2.41 stdc_bit_floor_ul F
+GLIBC_2.41 stdc_bit_floor_ull F
+GLIBC_2.41 stdc_bit_floor_us F
+GLIBC_2.41 stdc_bit_width_uc F
+GLIBC_2.41 stdc_bit_width_ui F
+GLIBC_2.41 stdc_bit_width_ul F
+GLIBC_2.41 stdc_bit_width_ull F
+GLIBC_2.41 stdc_bit_width_us F
+GLIBC_2.41 stdc_count_ones_uc F
+GLIBC_2.41 stdc_count_ones_ui F
+GLIBC_2.41 stdc_count_ones_ul F
+GLIBC_2.41 stdc_count_ones_ull F
+GLIBC_2.41 stdc_count_ones_us F
+GLIBC_2.41 stdc_count_zeros_uc F
+GLIBC_2.41 stdc_count_zeros_ui F
+GLIBC_2.41 stdc_count_zeros_ul F
+GLIBC_2.41 stdc_count_zeros_ull F
+GLIBC_2.41 stdc_count_zeros_us F
+GLIBC_2.41 stdc_first_leading_one_uc F
+GLIBC_2.41 stdc_first_leading_one_ui F
+GLIBC_2.41 stdc_first_leading_one_ul F
+GLIBC_2.41 stdc_first_leading_one_ull F
+GLIBC_2.41 stdc_first_leading_one_us F
+GLIBC_2.41 stdc_first_leading_zero_uc F
+GLIBC_2.41 stdc_first_leading_zero_ui F
+GLIBC_2.41 stdc_first_leading_zero_ul F
+GLIBC_2.41 stdc_first_leading_zero_ull F
+GLIBC_2.41 stdc_first_leading_zero_us F
+GLIBC_2.41 stdc_first_trailing_one_uc F
+GLIBC_2.41 stdc_first_trailing_one_ui F
+GLIBC_2.41 stdc_first_trailing_one_ul F
+GLIBC_2.41 stdc_first_trailing_one_ull F
+GLIBC_2.41 stdc_first_trailing_one_us F
+GLIBC_2.41 stdc_first_trailing_zero_uc F
+GLIBC_2.41 stdc_first_trailing_zero_ui F
+GLIBC_2.41 stdc_first_trailing_zero_ul F
+GLIBC_2.41 stdc_first_trailing_zero_ull F
+GLIBC_2.41 stdc_first_trailing_zero_us F
+GLIBC_2.41 stdc_has_single_bit_uc F
+GLIBC_2.41 stdc_has_single_bit_ui F
+GLIBC_2.41 stdc_has_single_bit_ul F
+GLIBC_2.41 stdc_has_single_bit_ull F
+GLIBC_2.41 stdc_has_single_bit_us F
+GLIBC_2.41 stdc_leading_ones_uc F
+GLIBC_2.41 stdc_leading_ones_ui F
+GLIBC_2.41 stdc_leading_ones_ul F
+GLIBC_2.41 stdc_leading_ones_ull F
+GLIBC_2.41 stdc_leading_ones_us F
+GLIBC_2.41 stdc_leading_zeros_uc F
+GLIBC_2.41 stdc_leading_zeros_ui F
+GLIBC_2.41 stdc_leading_zeros_ul F
+GLIBC_2.41 stdc_leading_zeros_ull F
+GLIBC_2.41 stdc_leading_zeros_us F
+GLIBC_2.41 stdc_trailing_ones_uc F
+GLIBC_2.41 stdc_trailing_ones_ui F
+GLIBC_2.41 stdc_trailing_ones_ul F
+GLIBC_2.41 stdc_trailing_ones_ull F
+GLIBC_2.41 stdc_trailing_ones_us F
+GLIBC_2.41 stdc_trailing_zeros_uc F
+GLIBC_2.41 stdc_trailing_zeros_ui F
+GLIBC_2.41 stdc_trailing_zeros_ul F
+GLIBC_2.41 stdc_trailing_zeros_ull F
+GLIBC_2.41 stdc_trailing_zeros_us F
+GLIBC_2.41 stderr D 0x4
+GLIBC_2.41 stdin D 0x4
+GLIBC_2.41 stdout D 0x4
+GLIBC_2.41 stpcpy F
+GLIBC_2.41 stpncpy F
+GLIBC_2.41 strcasecmp F
+GLIBC_2.41 strcasecmp_l F
+GLIBC_2.41 strcasestr F
+GLIBC_2.41 strcat F
+GLIBC_2.41 strchr F
+GLIBC_2.41 strchrnul F
+GLIBC_2.41 strcmp F
+GLIBC_2.41 strcoll F
+GLIBC_2.41 strcoll_l F
+GLIBC_2.41 strcpy F
+GLIBC_2.41 strcspn F
+GLIBC_2.41 strdup F
+GLIBC_2.41 strerror F
+GLIBC_2.41 strerror_l F
+GLIBC_2.41 strerror_r F
+GLIBC_2.41 strerrordesc_np F
+GLIBC_2.41 strerrorname_np F
+GLIBC_2.41 strfmon F
+GLIBC_2.41 strfmon_l F
+GLIBC_2.41 strfromd F
+GLIBC_2.41 strfromf F
+GLIBC_2.41 strfromf128 F
+GLIBC_2.41 strfromf32 F
+GLIBC_2.41 strfromf32x F
+GLIBC_2.41 strfromf64 F
+GLIBC_2.41 strfromf64x F
+GLIBC_2.41 strfroml F
+GLIBC_2.41 strfry F
+GLIBC_2.41 strftime F
+GLIBC_2.41 strftime_l F
+GLIBC_2.41 strlcat F
+GLIBC_2.41 strlcpy F
+GLIBC_2.41 strlen F
+GLIBC_2.41 strncasecmp F
+GLIBC_2.41 strncasecmp_l F
+GLIBC_2.41 strncat F
+GLIBC_2.41 strncmp F
+GLIBC_2.41 strncpy F
+GLIBC_2.41 strndup F
+GLIBC_2.41 strnlen F
+GLIBC_2.41 strpbrk F
+GLIBC_2.41 strptime F
+GLIBC_2.41 strptime_l F
+GLIBC_2.41 strrchr F
+GLIBC_2.41 strsep F
+GLIBC_2.41 strsignal F
+GLIBC_2.41 strspn F
+GLIBC_2.41 strstr F
+GLIBC_2.41 strtod F
+GLIBC_2.41 strtod_l F
+GLIBC_2.41 strtof F
+GLIBC_2.41 strtof128 F
+GLIBC_2.41 strtof128_l F
+GLIBC_2.41 strtof32 F
+GLIBC_2.41 strtof32_l F
+GLIBC_2.41 strtof32x F
+GLIBC_2.41 strtof32x_l F
+GLIBC_2.41 strtof64 F
+GLIBC_2.41 strtof64_l F
+GLIBC_2.41 strtof64x F
+GLIBC_2.41 strtof64x_l F
+GLIBC_2.41 strtof_l F
+GLIBC_2.41 strtoimax F
+GLIBC_2.41 strtok F
+GLIBC_2.41 strtok_r F
+GLIBC_2.41 strtol F
+GLIBC_2.41 strtol_l F
+GLIBC_2.41 strtold F
+GLIBC_2.41 strtold_l F
+GLIBC_2.41 strtoll F
+GLIBC_2.41 strtoll_l F
+GLIBC_2.41 strtoq F
+GLIBC_2.41 strtoul F
+GLIBC_2.41 strtoul_l F
+GLIBC_2.41 strtoull F
+GLIBC_2.41 strtoull_l F
+GLIBC_2.41 strtoumax F
+GLIBC_2.41 strtouq F
+GLIBC_2.41 strverscmp F
+GLIBC_2.41 strxfrm F
+GLIBC_2.41 strxfrm_l F
+GLIBC_2.41 stty F
+GLIBC_2.41 swab F
+GLIBC_2.41 swapcontext F
+GLIBC_2.41 swapoff F
+GLIBC_2.41 swapon F
+GLIBC_2.41 swprintf F
+GLIBC_2.41 swscanf F
+GLIBC_2.41 symlink F
+GLIBC_2.41 symlinkat F
+GLIBC_2.41 sync F
+GLIBC_2.41 sync_file_range F
+GLIBC_2.41 syncfs F
+GLIBC_2.41 syscall F
+GLIBC_2.41 sysconf F
+GLIBC_2.41 sysinfo F
+GLIBC_2.41 syslog F
+GLIBC_2.41 system F
+GLIBC_2.41 sysv_signal F
+GLIBC_2.41 tcdrain F
+GLIBC_2.41 tcflow F
+GLIBC_2.41 tcflush F
+GLIBC_2.41 tcgetattr F
+GLIBC_2.41 tcgetpgrp F
+GLIBC_2.41 tcgetsid F
+GLIBC_2.41 tcsendbreak F
+GLIBC_2.41 tcsetattr F
+GLIBC_2.41 tcsetpgrp F
+GLIBC_2.41 tdelete F
+GLIBC_2.41 tdestroy F
+GLIBC_2.41 tee F
+GLIBC_2.41 telldir F
+GLIBC_2.41 tempnam F
+GLIBC_2.41 textdomain F
+GLIBC_2.41 tfind F
+GLIBC_2.41 tgkill F
+GLIBC_2.41 thrd_create F
+GLIBC_2.41 thrd_current F
+GLIBC_2.41 thrd_detach F
+GLIBC_2.41 thrd_equal F
+GLIBC_2.41 thrd_exit F
+GLIBC_2.41 thrd_join F
+GLIBC_2.41 thrd_sleep F
+GLIBC_2.41 thrd_yield F
+GLIBC_2.41 time F
+GLIBC_2.41 timegm F
+GLIBC_2.41 timelocal F
+GLIBC_2.41 timer_create F
+GLIBC_2.41 timer_delete F
+GLIBC_2.41 timer_getoverrun F
+GLIBC_2.41 timer_gettime F
+GLIBC_2.41 timer_settime F
+GLIBC_2.41 timerfd_create F
+GLIBC_2.41 timerfd_gettime F
+GLIBC_2.41 timerfd_settime F
+GLIBC_2.41 times F
+GLIBC_2.41 timespec_get F
+GLIBC_2.41 timespec_getres F
+GLIBC_2.41 timezone D 0x4
+GLIBC_2.41 tmpfile F
+GLIBC_2.41 tmpfile64 F
+GLIBC_2.41 tmpnam F
+GLIBC_2.41 tmpnam_r F
+GLIBC_2.41 toascii F
+GLIBC_2.41 tolower F
+GLIBC_2.41 tolower_l F
+GLIBC_2.41 toupper F
+GLIBC_2.41 toupper_l F
+GLIBC_2.41 towctrans F
+GLIBC_2.41 towctrans_l F
+GLIBC_2.41 towlower F
+GLIBC_2.41 towlower_l F
+GLIBC_2.41 towupper F
+GLIBC_2.41 towupper_l F
+GLIBC_2.41 truncate F
+GLIBC_2.41 truncate64 F
+GLIBC_2.41 tsearch F
+GLIBC_2.41 tss_create F
+GLIBC_2.41 tss_delete F
+GLIBC_2.41 tss_get F
+GLIBC_2.41 tss_set F
+GLIBC_2.41 ttyname F
+GLIBC_2.41 ttyname_r F
+GLIBC_2.41 ttyslot F
+GLIBC_2.41 twalk F
+GLIBC_2.41 twalk_r F
+GLIBC_2.41 tzname D 0x8
+GLIBC_2.41 tzset F
+GLIBC_2.41 ualarm F
+GLIBC_2.41 ulckpwdf F
+GLIBC_2.41 ulimit F
+GLIBC_2.41 umask F
+GLIBC_2.41 umount F
+GLIBC_2.41 umount2 F
+GLIBC_2.41 uname F
+GLIBC_2.41 ungetc F
+GLIBC_2.41 ungetwc F
+GLIBC_2.41 unlink F
+GLIBC_2.41 unlinkat F
+GLIBC_2.41 unlockpt F
+GLIBC_2.41 unsetenv F
+GLIBC_2.41 unshare F
+GLIBC_2.41 updwtmp F
+GLIBC_2.41 updwtmpx F
+GLIBC_2.41 uselocale F
+GLIBC_2.41 usleep F
+GLIBC_2.41 utime F
+GLIBC_2.41 utimensat F
+GLIBC_2.41 utimes F
+GLIBC_2.41 utmpname F
+GLIBC_2.41 utmpxname F
+GLIBC_2.41 valloc F
+GLIBC_2.41 vasprintf F
+GLIBC_2.41 vdprintf F
+GLIBC_2.41 verr F
+GLIBC_2.41 verrx F
+GLIBC_2.41 versionsort F
+GLIBC_2.41 versionsort64 F
+GLIBC_2.41 vfork F
+GLIBC_2.41 vfprintf F
+GLIBC_2.41 vfscanf F
+GLIBC_2.41 vfwprintf F
+GLIBC_2.41 vfwscanf F
+GLIBC_2.41 vhangup F
+GLIBC_2.41 vlimit F
+GLIBC_2.41 vmsplice F
+GLIBC_2.41 vprintf F
+GLIBC_2.41 vscanf F
+GLIBC_2.41 vsnprintf F
+GLIBC_2.41 vsprintf F
+GLIBC_2.41 vsscanf F
+GLIBC_2.41 vswprintf F
+GLIBC_2.41 vswscanf F
+GLIBC_2.41 vsyslog F
+GLIBC_2.41 vwarn F
+GLIBC_2.41 vwarnx F
+GLIBC_2.41 vwprintf F
+GLIBC_2.41 vwscanf F
+GLIBC_2.41 wait F
+GLIBC_2.41 wait3 F
+GLIBC_2.41 wait4 F
+GLIBC_2.41 waitid F
+GLIBC_2.41 waitpid F
+GLIBC_2.41 warn F
+GLIBC_2.41 warnx F
+GLIBC_2.41 wcpcpy F
+GLIBC_2.41 wcpncpy F
+GLIBC_2.41 wcrtomb F
+GLIBC_2.41 wcscasecmp F
+GLIBC_2.41 wcscasecmp_l F
+GLIBC_2.41 wcscat F
+GLIBC_2.41 wcschr F
+GLIBC_2.41 wcschrnul F
+GLIBC_2.41 wcscmp F
+GLIBC_2.41 wcscoll F
+GLIBC_2.41 wcscoll_l F
+GLIBC_2.41 wcscpy F
+GLIBC_2.41 wcscspn F
+GLIBC_2.41 wcsdup F
+GLIBC_2.41 wcsftime F
+GLIBC_2.41 wcsftime_l F
+GLIBC_2.41 wcslcat F
+GLIBC_2.41 wcslcpy F
+GLIBC_2.41 wcslen F
+GLIBC_2.41 wcsncasecmp F
+GLIBC_2.41 wcsncasecmp_l F
+GLIBC_2.41 wcsncat F
+GLIBC_2.41 wcsncmp F
+GLIBC_2.41 wcsncpy F
+GLIBC_2.41 wcsnlen F
+GLIBC_2.41 wcsnrtombs F
+GLIBC_2.41 wcspbrk F
+GLIBC_2.41 wcsrchr F
+GLIBC_2.41 wcsrtombs F
+GLIBC_2.41 wcsspn F
+GLIBC_2.41 wcsstr F
+GLIBC_2.41 wcstod F
+GLIBC_2.41 wcstod_l F
+GLIBC_2.41 wcstof F
+GLIBC_2.41 wcstof128 F
+GLIBC_2.41 wcstof128_l F
+GLIBC_2.41 wcstof32 F
+GLIBC_2.41 wcstof32_l F
+GLIBC_2.41 wcstof32x F
+GLIBC_2.41 wcstof32x_l F
+GLIBC_2.41 wcstof64 F
+GLIBC_2.41 wcstof64_l F
+GLIBC_2.41 wcstof64x F
+GLIBC_2.41 wcstof64x_l F
+GLIBC_2.41 wcstof_l F
+GLIBC_2.41 wcstoimax F
+GLIBC_2.41 wcstok F
+GLIBC_2.41 wcstol F
+GLIBC_2.41 wcstol_l F
+GLIBC_2.41 wcstold F
+GLIBC_2.41 wcstold_l F
+GLIBC_2.41 wcstoll F
+GLIBC_2.41 wcstoll_l F
+GLIBC_2.41 wcstombs F
+GLIBC_2.41 wcstoq F
+GLIBC_2.41 wcstoul F
+GLIBC_2.41 wcstoul_l F
+GLIBC_2.41 wcstoull F
+GLIBC_2.41 wcstoull_l F
+GLIBC_2.41 wcstoumax F
+GLIBC_2.41 wcstouq F
+GLIBC_2.41 wcswcs F
+GLIBC_2.41 wcswidth F
+GLIBC_2.41 wcsxfrm F
+GLIBC_2.41 wcsxfrm_l F
+GLIBC_2.41 wctob F
+GLIBC_2.41 wctomb F
+GLIBC_2.41 wctrans F
+GLIBC_2.41 wctrans_l F
+GLIBC_2.41 wctype F
+GLIBC_2.41 wctype_l F
+GLIBC_2.41 wcwidth F
+GLIBC_2.41 wmemchr F
+GLIBC_2.41 wmemcmp F
+GLIBC_2.41 wmemcpy F
+GLIBC_2.41 wmemmove F
+GLIBC_2.41 wmempcpy F
+GLIBC_2.41 wmemset F
+GLIBC_2.41 wordexp F
+GLIBC_2.41 wordfree F
+GLIBC_2.41 wprintf F
+GLIBC_2.41 write F
+GLIBC_2.41 writev F
+GLIBC_2.41 wscanf F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libc_malloc_debug.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libc_malloc_debug.abilist
new file mode 100644
index 0000000000..d8d33a0658
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libc_malloc_debug.abilist
@@ -0,0 +1,26 @@
+GLIBC_2.41 __free_hook D 0x4
+GLIBC_2.41 __malloc_hook D 0x4
+GLIBC_2.41 __memalign_hook D 0x4
+GLIBC_2.41 __realloc_hook D 0x4
+GLIBC_2.41 aligned_alloc F
+GLIBC_2.41 calloc F
+GLIBC_2.41 free F
+GLIBC_2.41 mallinfo F
+GLIBC_2.41 mallinfo2 F
+GLIBC_2.41 malloc F
+GLIBC_2.41 malloc_info F
+GLIBC_2.41 malloc_stats F
+GLIBC_2.41 malloc_trim F
+GLIBC_2.41 malloc_usable_size F
+GLIBC_2.41 mallopt F
+GLIBC_2.41 mcheck F
+GLIBC_2.41 mcheck_check_all F
+GLIBC_2.41 mcheck_pedantic F
+GLIBC_2.41 memalign F
+GLIBC_2.41 mprobe F
+GLIBC_2.41 mtrace F
+GLIBC_2.41 muntrace F
+GLIBC_2.41 posix_memalign F
+GLIBC_2.41 pvalloc F
+GLIBC_2.41 realloc F
+GLIBC_2.41 valloc F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist
new file mode 100644
index 0000000000..e69de29bb2
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist
new file mode 100644
index 0000000000..4cd37d8c5e
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist
@@ -0,0 +1,1126 @@
+GLIBC_2.41 __clog10 F
+GLIBC_2.41 __clog10f F
+GLIBC_2.41 __clog10l F
+GLIBC_2.41 __finite F
+GLIBC_2.41 __finitef F
+GLIBC_2.41 __finitel F
+GLIBC_2.41 __fpclassify F
+GLIBC_2.41 __fpclassifyf F
+GLIBC_2.41 __fpclassifyl F
+GLIBC_2.41 __iseqsig F
+GLIBC_2.41 __iseqsigf F
+GLIBC_2.41 __iseqsigl F
+GLIBC_2.41 __issignaling F
+GLIBC_2.41 __issignalingf F
+GLIBC_2.41 __issignalingl F
+GLIBC_2.41 __signbit F
+GLIBC_2.41 __signbitf F
+GLIBC_2.41 __signbitl F
+GLIBC_2.41 __signgam D 0x4
+GLIBC_2.41 acos F
+GLIBC_2.41 acosf F
+GLIBC_2.41 acosf128 F
+GLIBC_2.41 acosf32 F
+GLIBC_2.41 acosf32x F
+GLIBC_2.41 acosf64 F
+GLIBC_2.41 acosf64x F
+GLIBC_2.41 acosh F
+GLIBC_2.41 acoshf F
+GLIBC_2.41 acoshf128 F
+GLIBC_2.41 acoshf32 F
+GLIBC_2.41 acoshf32x F
+GLIBC_2.41 acoshf64 F
+GLIBC_2.41 acoshf64x F
+GLIBC_2.41 acoshl F
+GLIBC_2.41 acosl F
+GLIBC_2.41 acospi F
+GLIBC_2.41 acospif F
+GLIBC_2.41 acospif128 F
+GLIBC_2.41 acospif32 F
+GLIBC_2.41 acospif32x F
+GLIBC_2.41 acospif64 F
+GLIBC_2.41 acospif64x F
+GLIBC_2.41 acospil F
+GLIBC_2.41 asin F
+GLIBC_2.41 asinf F
+GLIBC_2.41 asinf128 F
+GLIBC_2.41 asinf32 F
+GLIBC_2.41 asinf32x F
+GLIBC_2.41 asinf64 F
+GLIBC_2.41 asinf64x F
+GLIBC_2.41 asinh F
+GLIBC_2.41 asinhf F
+GLIBC_2.41 asinhf128 F
+GLIBC_2.41 asinhf32 F
+GLIBC_2.41 asinhf32x F
+GLIBC_2.41 asinhf64 F
+GLIBC_2.41 asinhf64x F
+GLIBC_2.41 asinhl F
+GLIBC_2.41 asinl F
+GLIBC_2.41 asinpi F
+GLIBC_2.41 asinpif F
+GLIBC_2.41 asinpif128 F
+GLIBC_2.41 asinpif32 F
+GLIBC_2.41 asinpif32x F
+GLIBC_2.41 asinpif64 F
+GLIBC_2.41 asinpif64x F
+GLIBC_2.41 asinpil F
+GLIBC_2.41 atan F
+GLIBC_2.41 atan2 F
+GLIBC_2.41 atan2f F
+GLIBC_2.41 atan2f128 F
+GLIBC_2.41 atan2f32 F
+GLIBC_2.41 atan2f32x F
+GLIBC_2.41 atan2f64 F
+GLIBC_2.41 atan2f64x F
+GLIBC_2.41 atan2l F
+GLIBC_2.41 atan2pi F
+GLIBC_2.41 atan2pif F
+GLIBC_2.41 atan2pif128 F
+GLIBC_2.41 atan2pif32 F
+GLIBC_2.41 atan2pif32x F
+GLIBC_2.41 atan2pif64 F
+GLIBC_2.41 atan2pif64x F
+GLIBC_2.41 atan2pil F
+GLIBC_2.41 atanf F
+GLIBC_2.41 atanf128 F
+GLIBC_2.41 atanf32 F
+GLIBC_2.41 atanf32x F
+GLIBC_2.41 atanf64 F
+GLIBC_2.41 atanf64x F
+GLIBC_2.41 atanh F
+GLIBC_2.41 atanhf F
+GLIBC_2.41 atanhf128 F
+GLIBC_2.41 atanhf32 F
+GLIBC_2.41 atanhf32x F
+GLIBC_2.41 atanhf64 F
+GLIBC_2.41 atanhf64x F
+GLIBC_2.41 atanhl F
+GLIBC_2.41 atanl F
+GLIBC_2.41 atanpi F
+GLIBC_2.41 atanpif F
+GLIBC_2.41 atanpif128 F
+GLIBC_2.41 atanpif32 F
+GLIBC_2.41 atanpif32x F
+GLIBC_2.41 atanpif64 F
+GLIBC_2.41 atanpif64x F
+GLIBC_2.41 atanpil F
+GLIBC_2.41 cabs F
+GLIBC_2.41 cabsf F
+GLIBC_2.41 cabsf128 F
+GLIBC_2.41 cabsf32 F
+GLIBC_2.41 cabsf32x F
+GLIBC_2.41 cabsf64 F
+GLIBC_2.41 cabsf64x F
+GLIBC_2.41 cabsl F
+GLIBC_2.41 cacos F
+GLIBC_2.41 cacosf F
+GLIBC_2.41 cacosf128 F
+GLIBC_2.41 cacosf32 F
+GLIBC_2.41 cacosf32x F
+GLIBC_2.41 cacosf64 F
+GLIBC_2.41 cacosf64x F
+GLIBC_2.41 cacosh F
+GLIBC_2.41 cacoshf F
+GLIBC_2.41 cacoshf128 F
+GLIBC_2.41 cacoshf32 F
+GLIBC_2.41 cacoshf32x F
+GLIBC_2.41 cacoshf64 F
+GLIBC_2.41 cacoshf64x F
+GLIBC_2.41 cacoshl F
+GLIBC_2.41 cacosl F
+GLIBC_2.41 canonicalize F
+GLIBC_2.41 canonicalizef F
+GLIBC_2.41 canonicalizef128 F
+GLIBC_2.41 canonicalizef32 F
+GLIBC_2.41 canonicalizef32x F
+GLIBC_2.41 canonicalizef64 F
+GLIBC_2.41 canonicalizef64x F
+GLIBC_2.41 canonicalizel F
+GLIBC_2.41 carg F
+GLIBC_2.41 cargf F
+GLIBC_2.41 cargf128 F
+GLIBC_2.41 cargf32 F
+GLIBC_2.41 cargf32x F
+GLIBC_2.41 cargf64 F
+GLIBC_2.41 cargf64x F
+GLIBC_2.41 cargl F
+GLIBC_2.41 casin F
+GLIBC_2.41 casinf F
+GLIBC_2.41 casinf128 F
+GLIBC_2.41 casinf32 F
+GLIBC_2.41 casinf32x F
+GLIBC_2.41 casinf64 F
+GLIBC_2.41 casinf64x F
+GLIBC_2.41 casinh F
+GLIBC_2.41 casinhf F
+GLIBC_2.41 casinhf128 F
+GLIBC_2.41 casinhf32 F
+GLIBC_2.41 casinhf32x F
+GLIBC_2.41 casinhf64 F
+GLIBC_2.41 casinhf64x F
+GLIBC_2.41 casinhl F
+GLIBC_2.41 casinl F
+GLIBC_2.41 catan F
+GLIBC_2.41 catanf F
+GLIBC_2.41 catanf128 F
+GLIBC_2.41 catanf32 F
+GLIBC_2.41 catanf32x F
+GLIBC_2.41 catanf64 F
+GLIBC_2.41 catanf64x F
+GLIBC_2.41 catanh F
+GLIBC_2.41 catanhf F
+GLIBC_2.41 catanhf128 F
+GLIBC_2.41 catanhf32 F
+GLIBC_2.41 catanhf32x F
+GLIBC_2.41 catanhf64 F
+GLIBC_2.41 catanhf64x F
+GLIBC_2.41 catanhl F
+GLIBC_2.41 catanl F
+GLIBC_2.41 cbrt F
+GLIBC_2.41 cbrtf F
+GLIBC_2.41 cbrtf128 F
+GLIBC_2.41 cbrtf32 F
+GLIBC_2.41 cbrtf32x F
+GLIBC_2.41 cbrtf64 F
+GLIBC_2.41 cbrtf64x F
+GLIBC_2.41 cbrtl F
+GLIBC_2.41 ccos F
+GLIBC_2.41 ccosf F
+GLIBC_2.41 ccosf128 F
+GLIBC_2.41 ccosf32 F
+GLIBC_2.41 ccosf32x F
+GLIBC_2.41 ccosf64 F
+GLIBC_2.41 ccosf64x F
+GLIBC_2.41 ccosh F
+GLIBC_2.41 ccoshf F
+GLIBC_2.41 ccoshf128 F
+GLIBC_2.41 ccoshf32 F
+GLIBC_2.41 ccoshf32x F
+GLIBC_2.41 ccoshf64 F
+GLIBC_2.41 ccoshf64x F
+GLIBC_2.41 ccoshl F
+GLIBC_2.41 ccosl F
+GLIBC_2.41 ceil F
+GLIBC_2.41 ceilf F
+GLIBC_2.41 ceilf128 F
+GLIBC_2.41 ceilf32 F
+GLIBC_2.41 ceilf32x F
+GLIBC_2.41 ceilf64 F
+GLIBC_2.41 ceilf64x F
+GLIBC_2.41 ceill F
+GLIBC_2.41 cexp F
+GLIBC_2.41 cexpf F
+GLIBC_2.41 cexpf128 F
+GLIBC_2.41 cexpf32 F
+GLIBC_2.41 cexpf32x F
+GLIBC_2.41 cexpf64 F
+GLIBC_2.41 cexpf64x F
+GLIBC_2.41 cexpl F
+GLIBC_2.41 cimag F
+GLIBC_2.41 cimagf F
+GLIBC_2.41 cimagf128 F
+GLIBC_2.41 cimagf32 F
+GLIBC_2.41 cimagf32x F
+GLIBC_2.41 cimagf64 F
+GLIBC_2.41 cimagf64x F
+GLIBC_2.41 cimagl F
+GLIBC_2.41 clog F
+GLIBC_2.41 clog10 F
+GLIBC_2.41 clog10f F
+GLIBC_2.41 clog10f128 F
+GLIBC_2.41 clog10f32 F
+GLIBC_2.41 clog10f32x F
+GLIBC_2.41 clog10f64 F
+GLIBC_2.41 clog10f64x F
+GLIBC_2.41 clog10l F
+GLIBC_2.41 clogf F
+GLIBC_2.41 clogf128 F
+GLIBC_2.41 clogf32 F
+GLIBC_2.41 clogf32x F
+GLIBC_2.41 clogf64 F
+GLIBC_2.41 clogf64x F
+GLIBC_2.41 clogl F
+GLIBC_2.41 conj F
+GLIBC_2.41 conjf F
+GLIBC_2.41 conjf128 F
+GLIBC_2.41 conjf32 F
+GLIBC_2.41 conjf32x F
+GLIBC_2.41 conjf64 F
+GLIBC_2.41 conjf64x F
+GLIBC_2.41 conjl F
+GLIBC_2.41 copysign F
+GLIBC_2.41 copysignf F
+GLIBC_2.41 copysignf128 F
+GLIBC_2.41 copysignf32 F
+GLIBC_2.41 copysignf32x F
+GLIBC_2.41 copysignf64 F
+GLIBC_2.41 copysignf64x F
+GLIBC_2.41 copysignl F
+GLIBC_2.41 cos F
+GLIBC_2.41 cosf F
+GLIBC_2.41 cosf128 F
+GLIBC_2.41 cosf32 F
+GLIBC_2.41 cosf32x F
+GLIBC_2.41 cosf64 F
+GLIBC_2.41 cosf64x F
+GLIBC_2.41 cosh F
+GLIBC_2.41 coshf F
+GLIBC_2.41 coshf128 F
+GLIBC_2.41 coshf32 F
+GLIBC_2.41 coshf32x F
+GLIBC_2.41 coshf64 F
+GLIBC_2.41 coshf64x F
+GLIBC_2.41 coshl F
+GLIBC_2.41 cosl F
+GLIBC_2.41 cospi F
+GLIBC_2.41 cospif F
+GLIBC_2.41 cospif128 F
+GLIBC_2.41 cospif32 F
+GLIBC_2.41 cospif32x F
+GLIBC_2.41 cospif64 F
+GLIBC_2.41 cospif64x F
+GLIBC_2.41 cospil F
+GLIBC_2.41 cpow F
+GLIBC_2.41 cpowf F
+GLIBC_2.41 cpowf128 F
+GLIBC_2.41 cpowf32 F
+GLIBC_2.41 cpowf32x F
+GLIBC_2.41 cpowf64 F
+GLIBC_2.41 cpowf64x F
+GLIBC_2.41 cpowl F
+GLIBC_2.41 cproj F
+GLIBC_2.41 cprojf F
+GLIBC_2.41 cprojf128 F
+GLIBC_2.41 cprojf32 F
+GLIBC_2.41 cprojf32x F
+GLIBC_2.41 cprojf64 F
+GLIBC_2.41 cprojf64x F
+GLIBC_2.41 cprojl F
+GLIBC_2.41 creal F
+GLIBC_2.41 crealf F
+GLIBC_2.41 crealf128 F
+GLIBC_2.41 crealf32 F
+GLIBC_2.41 crealf32x F
+GLIBC_2.41 crealf64 F
+GLIBC_2.41 crealf64x F
+GLIBC_2.41 creall F
+GLIBC_2.41 csin F
+GLIBC_2.41 csinf F
+GLIBC_2.41 csinf128 F
+GLIBC_2.41 csinf32 F
+GLIBC_2.41 csinf32x F
+GLIBC_2.41 csinf64 F
+GLIBC_2.41 csinf64x F
+GLIBC_2.41 csinh F
+GLIBC_2.41 csinhf F
+GLIBC_2.41 csinhf128 F
+GLIBC_2.41 csinhf32 F
+GLIBC_2.41 csinhf32x F
+GLIBC_2.41 csinhf64 F
+GLIBC_2.41 csinhf64x F
+GLIBC_2.41 csinhl F
+GLIBC_2.41 csinl F
+GLIBC_2.41 csqrt F
+GLIBC_2.41 csqrtf F
+GLIBC_2.41 csqrtf128 F
+GLIBC_2.41 csqrtf32 F
+GLIBC_2.41 csqrtf32x F
+GLIBC_2.41 csqrtf64 F
+GLIBC_2.41 csqrtf64x F
+GLIBC_2.41 csqrtl F
+GLIBC_2.41 ctan F
+GLIBC_2.41 ctanf F
+GLIBC_2.41 ctanf128 F
+GLIBC_2.41 ctanf32 F
+GLIBC_2.41 ctanf32x F
+GLIBC_2.41 ctanf64 F
+GLIBC_2.41 ctanf64x F
+GLIBC_2.41 ctanh F
+GLIBC_2.41 ctanhf F
+GLIBC_2.41 ctanhf128 F
+GLIBC_2.41 ctanhf32 F
+GLIBC_2.41 ctanhf32x F
+GLIBC_2.41 ctanhf64 F
+GLIBC_2.41 ctanhf64x F
+GLIBC_2.41 ctanhl F
+GLIBC_2.41 ctanl F
+GLIBC_2.41 daddl F
+GLIBC_2.41 ddivl F
+GLIBC_2.41 dfmal F
+GLIBC_2.41 dmull F
+GLIBC_2.41 drem F
+GLIBC_2.41 dremf F
+GLIBC_2.41 dreml F
+GLIBC_2.41 dsqrtl F
+GLIBC_2.41 dsubl F
+GLIBC_2.41 erf F
+GLIBC_2.41 erfc F
+GLIBC_2.41 erfcf F
+GLIBC_2.41 erfcf128 F
+GLIBC_2.41 erfcf32 F
+GLIBC_2.41 erfcf32x F
+GLIBC_2.41 erfcf64 F
+GLIBC_2.41 erfcf64x F
+GLIBC_2.41 erfcl F
+GLIBC_2.41 erff F
+GLIBC_2.41 erff128 F
+GLIBC_2.41 erff32 F
+GLIBC_2.41 erff32x F
+GLIBC_2.41 erff64 F
+GLIBC_2.41 erff64x F
+GLIBC_2.41 erfl F
+GLIBC_2.41 exp F
+GLIBC_2.41 exp10 F
+GLIBC_2.41 exp10f F
+GLIBC_2.41 exp10f128 F
+GLIBC_2.41 exp10f32 F
+GLIBC_2.41 exp10f32x F
+GLIBC_2.41 exp10f64 F
+GLIBC_2.41 exp10f64x F
+GLIBC_2.41 exp10l F
+GLIBC_2.41 exp10m1 F
+GLIBC_2.41 exp10m1f F
+GLIBC_2.41 exp10m1f128 F
+GLIBC_2.41 exp10m1f32 F
+GLIBC_2.41 exp10m1f32x F
+GLIBC_2.41 exp10m1f64 F
+GLIBC_2.41 exp10m1f64x F
+GLIBC_2.41 exp10m1l F
+GLIBC_2.41 exp2 F
+GLIBC_2.41 exp2f F
+GLIBC_2.41 exp2f128 F
+GLIBC_2.41 exp2f32 F
+GLIBC_2.41 exp2f32x F
+GLIBC_2.41 exp2f64 F
+GLIBC_2.41 exp2f64x F
+GLIBC_2.41 exp2l F
+GLIBC_2.41 exp2m1 F
+GLIBC_2.41 exp2m1f F
+GLIBC_2.41 exp2m1f128 F
+GLIBC_2.41 exp2m1f32 F
+GLIBC_2.41 exp2m1f32x F
+GLIBC_2.41 exp2m1f64 F
+GLIBC_2.41 exp2m1f64x F
+GLIBC_2.41 exp2m1l F
+GLIBC_2.41 expf F
+GLIBC_2.41 expf128 F
+GLIBC_2.41 expf32 F
+GLIBC_2.41 expf32x F
+GLIBC_2.41 expf64 F
+GLIBC_2.41 expf64x F
+GLIBC_2.41 expl F
+GLIBC_2.41 expm1 F
+GLIBC_2.41 expm1f F
+GLIBC_2.41 expm1f128 F
+GLIBC_2.41 expm1f32 F
+GLIBC_2.41 expm1f32x F
+GLIBC_2.41 expm1f64 F
+GLIBC_2.41 expm1f64x F
+GLIBC_2.41 expm1l F
+GLIBC_2.41 f32addf128 F
+GLIBC_2.41 f32addf32x F
+GLIBC_2.41 f32addf64 F
+GLIBC_2.41 f32addf64x F
+GLIBC_2.41 f32divf128 F
+GLIBC_2.41 f32divf32x F
+GLIBC_2.41 f32divf64 F
+GLIBC_2.41 f32divf64x F
+GLIBC_2.41 f32fmaf128 F
+GLIBC_2.41 f32fmaf32x F
+GLIBC_2.41 f32fmaf64 F
+GLIBC_2.41 f32fmaf64x F
+GLIBC_2.41 f32mulf128 F
+GLIBC_2.41 f32mulf32x F
+GLIBC_2.41 f32mulf64 F
+GLIBC_2.41 f32mulf64x F
+GLIBC_2.41 f32sqrtf128 F
+GLIBC_2.41 f32sqrtf32x F
+GLIBC_2.41 f32sqrtf64 F
+GLIBC_2.41 f32sqrtf64x F
+GLIBC_2.41 f32subf128 F
+GLIBC_2.41 f32subf32x F
+GLIBC_2.41 f32subf64 F
+GLIBC_2.41 f32subf64x F
+GLIBC_2.41 f32xaddf128 F
+GLIBC_2.41 f32xaddf64 F
+GLIBC_2.41 f32xaddf64x F
+GLIBC_2.41 f32xdivf128 F
+GLIBC_2.41 f32xdivf64 F
+GLIBC_2.41 f32xdivf64x F
+GLIBC_2.41 f32xfmaf128 F
+GLIBC_2.41 f32xfmaf64 F
+GLIBC_2.41 f32xfmaf64x F
+GLIBC_2.41 f32xmulf128 F
+GLIBC_2.41 f32xmulf64 F
+GLIBC_2.41 f32xmulf64x F
+GLIBC_2.41 f32xsqrtf128 F
+GLIBC_2.41 f32xsqrtf64 F
+GLIBC_2.41 f32xsqrtf64x F
+GLIBC_2.41 f32xsubf128 F
+GLIBC_2.41 f32xsubf64 F
+GLIBC_2.41 f32xsubf64x F
+GLIBC_2.41 f64addf128 F
+GLIBC_2.41 f64addf64x F
+GLIBC_2.41 f64divf128 F
+GLIBC_2.41 f64divf64x F
+GLIBC_2.41 f64fmaf128 F
+GLIBC_2.41 f64fmaf64x F
+GLIBC_2.41 f64mulf128 F
+GLIBC_2.41 f64mulf64x F
+GLIBC_2.41 f64sqrtf128 F
+GLIBC_2.41 f64sqrtf64x F
+GLIBC_2.41 f64subf128 F
+GLIBC_2.41 f64subf64x F
+GLIBC_2.41 f64xaddf128 F
+GLIBC_2.41 f64xdivf128 F
+GLIBC_2.41 f64xfmaf128 F
+GLIBC_2.41 f64xmulf128 F
+GLIBC_2.41 f64xsqrtf128 F
+GLIBC_2.41 f64xsubf128 F
+GLIBC_2.41 fabs F
+GLIBC_2.41 fabsf F
+GLIBC_2.41 fabsf128 F
+GLIBC_2.41 fabsf32 F
+GLIBC_2.41 fabsf32x F
+GLIBC_2.41 fabsf64 F
+GLIBC_2.41 fabsf64x F
+GLIBC_2.41 fabsl F
+GLIBC_2.41 fadd F
+GLIBC_2.41 faddl F
+GLIBC_2.41 fdim F
+GLIBC_2.41 fdimf F
+GLIBC_2.41 fdimf128 F
+GLIBC_2.41 fdimf32 F
+GLIBC_2.41 fdimf32x F
+GLIBC_2.41 fdimf64 F
+GLIBC_2.41 fdimf64x F
+GLIBC_2.41 fdiml F
+GLIBC_2.41 fdiv F
+GLIBC_2.41 fdivl F
+GLIBC_2.41 feclearexcept F
+GLIBC_2.41 fedisableexcept F
+GLIBC_2.41 feenableexcept F
+GLIBC_2.41 fegetenv F
+GLIBC_2.41 fegetexcept F
+GLIBC_2.41 fegetexceptflag F
+GLIBC_2.41 fegetmode F
+GLIBC_2.41 fegetround F
+GLIBC_2.41 feholdexcept F
+GLIBC_2.41 feraiseexcept F
+GLIBC_2.41 fesetenv F
+GLIBC_2.41 fesetexcept F
+GLIBC_2.41 fesetexceptflag F
+GLIBC_2.41 fesetmode F
+GLIBC_2.41 fesetround F
+GLIBC_2.41 fetestexcept F
+GLIBC_2.41 fetestexceptflag F
+GLIBC_2.41 feupdateenv F
+GLIBC_2.41 ffma F
+GLIBC_2.41 ffmal F
+GLIBC_2.41 finite F
+GLIBC_2.41 finitef F
+GLIBC_2.41 finitel F
+GLIBC_2.41 floor F
+GLIBC_2.41 floorf F
+GLIBC_2.41 floorf128 F
+GLIBC_2.41 floorf32 F
+GLIBC_2.41 floorf32x F
+GLIBC_2.41 floorf64 F
+GLIBC_2.41 floorf64x F
+GLIBC_2.41 floorl F
+GLIBC_2.41 fma F
+GLIBC_2.41 fmaf F
+GLIBC_2.41 fmaf128 F
+GLIBC_2.41 fmaf32 F
+GLIBC_2.41 fmaf32x F
+GLIBC_2.41 fmaf64 F
+GLIBC_2.41 fmaf64x F
+GLIBC_2.41 fmal F
+GLIBC_2.41 fmax F
+GLIBC_2.41 fmaxf F
+GLIBC_2.41 fmaxf128 F
+GLIBC_2.41 fmaxf32 F
+GLIBC_2.41 fmaxf32x F
+GLIBC_2.41 fmaxf64 F
+GLIBC_2.41 fmaxf64x F
+GLIBC_2.41 fmaximum F
+GLIBC_2.41 fmaximum_mag F
+GLIBC_2.41 fmaximum_mag_num F
+GLIBC_2.41 fmaximum_mag_numf F
+GLIBC_2.41 fmaximum_mag_numf128 F
+GLIBC_2.41 fmaximum_mag_numf32 F
+GLIBC_2.41 fmaximum_mag_numf32x F
+GLIBC_2.41 fmaximum_mag_numf64 F
+GLIBC_2.41 fmaximum_mag_numf64x F
+GLIBC_2.41 fmaximum_mag_numl F
+GLIBC_2.41 fmaximum_magf F
+GLIBC_2.41 fmaximum_magf128 F
+GLIBC_2.41 fmaximum_magf32 F
+GLIBC_2.41 fmaximum_magf32x F
+GLIBC_2.41 fmaximum_magf64 F
+GLIBC_2.41 fmaximum_magf64x F
+GLIBC_2.41 fmaximum_magl F
+GLIBC_2.41 fmaximum_num F
+GLIBC_2.41 fmaximum_numf F
+GLIBC_2.41 fmaximum_numf128 F
+GLIBC_2.41 fmaximum_numf32 F
+GLIBC_2.41 fmaximum_numf32x F
+GLIBC_2.41 fmaximum_numf64 F
+GLIBC_2.41 fmaximum_numf64x F
+GLIBC_2.41 fmaximum_numl F
+GLIBC_2.41 fmaximumf F
+GLIBC_2.41 fmaximumf128 F
+GLIBC_2.41 fmaximumf32 F
+GLIBC_2.41 fmaximumf32x F
+GLIBC_2.41 fmaximumf64 F
+GLIBC_2.41 fmaximumf64x F
+GLIBC_2.41 fmaximuml F
+GLIBC_2.41 fmaxl F
+GLIBC_2.41 fmaxmag F
+GLIBC_2.41 fmaxmagf F
+GLIBC_2.41 fmaxmagf128 F
+GLIBC_2.41 fmaxmagf32 F
+GLIBC_2.41 fmaxmagf32x F
+GLIBC_2.41 fmaxmagf64 F
+GLIBC_2.41 fmaxmagf64x F
+GLIBC_2.41 fmaxmagl F
+GLIBC_2.41 fmin F
+GLIBC_2.41 fminf F
+GLIBC_2.41 fminf128 F
+GLIBC_2.41 fminf32 F
+GLIBC_2.41 fminf32x F
+GLIBC_2.41 fminf64 F
+GLIBC_2.41 fminf64x F
+GLIBC_2.41 fminimum F
+GLIBC_2.41 fminimum_mag F
+GLIBC_2.41 fminimum_mag_num F
+GLIBC_2.41 fminimum_mag_numf F
+GLIBC_2.41 fminimum_mag_numf128 F
+GLIBC_2.41 fminimum_mag_numf32 F
+GLIBC_2.41 fminimum_mag_numf32x F
+GLIBC_2.41 fminimum_mag_numf64 F
+GLIBC_2.41 fminimum_mag_numf64x F
+GLIBC_2.41 fminimum_mag_numl F
+GLIBC_2.41 fminimum_magf F
+GLIBC_2.41 fminimum_magf128 F
+GLIBC_2.41 fminimum_magf32 F
+GLIBC_2.41 fminimum_magf32x F
+GLIBC_2.41 fminimum_magf64 F
+GLIBC_2.41 fminimum_magf64x F
+GLIBC_2.41 fminimum_magl F
+GLIBC_2.41 fminimum_num F
+GLIBC_2.41 fminimum_numf F
+GLIBC_2.41 fminimum_numf128 F
+GLIBC_2.41 fminimum_numf32 F
+GLIBC_2.41 fminimum_numf32x F
+GLIBC_2.41 fminimum_numf64 F
+GLIBC_2.41 fminimum_numf64x F
+GLIBC_2.41 fminimum_numl F
+GLIBC_2.41 fminimumf F
+GLIBC_2.41 fminimumf128 F
+GLIBC_2.41 fminimumf32 F
+GLIBC_2.41 fminimumf32x F
+GLIBC_2.41 fminimumf64 F
+GLIBC_2.41 fminimumf64x F
+GLIBC_2.41 fminimuml F
+GLIBC_2.41 fminl F
+GLIBC_2.41 fminmag F
+GLIBC_2.41 fminmagf F
+GLIBC_2.41 fminmagf128 F
+GLIBC_2.41 fminmagf32 F
+GLIBC_2.41 fminmagf32x F
+GLIBC_2.41 fminmagf64 F
+GLIBC_2.41 fminmagf64x F
+GLIBC_2.41 fminmagl F
+GLIBC_2.41 fmod F
+GLIBC_2.41 fmodf F
+GLIBC_2.41 fmodf128 F
+GLIBC_2.41 fmodf32 F
+GLIBC_2.41 fmodf32x F
+GLIBC_2.41 fmodf64 F
+GLIBC_2.41 fmodf64x F
+GLIBC_2.41 fmodl F
+GLIBC_2.41 fmul F
+GLIBC_2.41 fmull F
+GLIBC_2.41 frexp F
+GLIBC_2.41 frexpf F
+GLIBC_2.41 frexpf128 F
+GLIBC_2.41 frexpf32 F
+GLIBC_2.41 frexpf32x F
+GLIBC_2.41 frexpf64 F
+GLIBC_2.41 frexpf64x F
+GLIBC_2.41 frexpl F
+GLIBC_2.41 fromfp F
+GLIBC_2.41 fromfpf F
+GLIBC_2.41 fromfpf128 F
+GLIBC_2.41 fromfpf32 F
+GLIBC_2.41 fromfpf32x F
+GLIBC_2.41 fromfpf64 F
+GLIBC_2.41 fromfpf64x F
+GLIBC_2.41 fromfpl F
+GLIBC_2.41 fromfpx F
+GLIBC_2.41 fromfpxf F
+GLIBC_2.41 fromfpxf128 F
+GLIBC_2.41 fromfpxf32 F
+GLIBC_2.41 fromfpxf32x F
+GLIBC_2.41 fromfpxf64 F
+GLIBC_2.41 fromfpxf64x F
+GLIBC_2.41 fromfpxl F
+GLIBC_2.41 fsqrt F
+GLIBC_2.41 fsqrtl F
+GLIBC_2.41 fsub F
+GLIBC_2.41 fsubl F
+GLIBC_2.41 gamma F
+GLIBC_2.41 gammaf F
+GLIBC_2.41 gammal F
+GLIBC_2.41 getpayload F
+GLIBC_2.41 getpayloadf F
+GLIBC_2.41 getpayloadf128 F
+GLIBC_2.41 getpayloadf32 F
+GLIBC_2.41 getpayloadf32x F
+GLIBC_2.41 getpayloadf64 F
+GLIBC_2.41 getpayloadf64x F
+GLIBC_2.41 getpayloadl F
+GLIBC_2.41 hypot F
+GLIBC_2.41 hypotf F
+GLIBC_2.41 hypotf128 F
+GLIBC_2.41 hypotf32 F
+GLIBC_2.41 hypotf32x F
+GLIBC_2.41 hypotf64 F
+GLIBC_2.41 hypotf64x F
+GLIBC_2.41 hypotl F
+GLIBC_2.41 ilogb F
+GLIBC_2.41 ilogbf F
+GLIBC_2.41 ilogbf128 F
+GLIBC_2.41 ilogbf32 F
+GLIBC_2.41 ilogbf32x F
+GLIBC_2.41 ilogbf64 F
+GLIBC_2.41 ilogbf64x F
+GLIBC_2.41 ilogbl F
+GLIBC_2.41 j0 F
+GLIBC_2.41 j0f F
+GLIBC_2.41 j0f128 F
+GLIBC_2.41 j0f32 F
+GLIBC_2.41 j0f32x F
+GLIBC_2.41 j0f64 F
+GLIBC_2.41 j0f64x F
+GLIBC_2.41 j0l F
+GLIBC_2.41 j1 F
+GLIBC_2.41 j1f F
+GLIBC_2.41 j1f128 F
+GLIBC_2.41 j1f32 F
+GLIBC_2.41 j1f32x F
+GLIBC_2.41 j1f64 F
+GLIBC_2.41 j1f64x F
+GLIBC_2.41 j1l F
+GLIBC_2.41 jn F
+GLIBC_2.41 jnf F
+GLIBC_2.41 jnf128 F
+GLIBC_2.41 jnf32 F
+GLIBC_2.41 jnf32x F
+GLIBC_2.41 jnf64 F
+GLIBC_2.41 jnf64x F
+GLIBC_2.41 jnl F
+GLIBC_2.41 ldexp F
+GLIBC_2.41 ldexpf F
+GLIBC_2.41 ldexpf128 F
+GLIBC_2.41 ldexpf32 F
+GLIBC_2.41 ldexpf32x F
+GLIBC_2.41 ldexpf64 F
+GLIBC_2.41 ldexpf64x F
+GLIBC_2.41 ldexpl F
+GLIBC_2.41 lgamma F
+GLIBC_2.41 lgamma_r F
+GLIBC_2.41 lgammaf F
+GLIBC_2.41 lgammaf128 F
+GLIBC_2.41 lgammaf128_r F
+GLIBC_2.41 lgammaf32 F
+GLIBC_2.41 lgammaf32_r F
+GLIBC_2.41 lgammaf32x F
+GLIBC_2.41 lgammaf32x_r F
+GLIBC_2.41 lgammaf64 F
+GLIBC_2.41 lgammaf64_r F
+GLIBC_2.41 lgammaf64x F
+GLIBC_2.41 lgammaf64x_r F
+GLIBC_2.41 lgammaf_r F
+GLIBC_2.41 lgammal F
+GLIBC_2.41 lgammal_r F
+GLIBC_2.41 llogb F
+GLIBC_2.41 llogbf F
+GLIBC_2.41 llogbf128 F
+GLIBC_2.41 llogbf32 F
+GLIBC_2.41 llogbf32x F
+GLIBC_2.41 llogbf64 F
+GLIBC_2.41 llogbf64x F
+GLIBC_2.41 llogbl F
+GLIBC_2.41 llrint F
+GLIBC_2.41 llrintf F
+GLIBC_2.41 llrintf128 F
+GLIBC_2.41 llrintf32 F
+GLIBC_2.41 llrintf32x F
+GLIBC_2.41 llrintf64 F
+GLIBC_2.41 llrintf64x F
+GLIBC_2.41 llrintl F
+GLIBC_2.41 llround F
+GLIBC_2.41 llroundf F
+GLIBC_2.41 llroundf128 F
+GLIBC_2.41 llroundf32 F
+GLIBC_2.41 llroundf32x F
+GLIBC_2.41 llroundf64 F
+GLIBC_2.41 llroundf64x F
+GLIBC_2.41 llroundl F
+GLIBC_2.41 log F
+GLIBC_2.41 log10 F
+GLIBC_2.41 log10f F
+GLIBC_2.41 log10f128 F
+GLIBC_2.41 log10f32 F
+GLIBC_2.41 log10f32x F
+GLIBC_2.41 log10f64 F
+GLIBC_2.41 log10f64x F
+GLIBC_2.41 log10l F
+GLIBC_2.41 log10p1 F
+GLIBC_2.41 log10p1f F
+GLIBC_2.41 log10p1f128 F
+GLIBC_2.41 log10p1f32 F
+GLIBC_2.41 log10p1f32x F
+GLIBC_2.41 log10p1f64 F
+GLIBC_2.41 log10p1f64x F
+GLIBC_2.41 log10p1l F
+GLIBC_2.41 log1p F
+GLIBC_2.41 log1pf F
+GLIBC_2.41 log1pf128 F
+GLIBC_2.41 log1pf32 F
+GLIBC_2.41 log1pf32x F
+GLIBC_2.41 log1pf64 F
+GLIBC_2.41 log1pf64x F
+GLIBC_2.41 log1pl F
+GLIBC_2.41 log2 F
+GLIBC_2.41 log2f F
+GLIBC_2.41 log2f128 F
+GLIBC_2.41 log2f32 F
+GLIBC_2.41 log2f32x F
+GLIBC_2.41 log2f64 F
+GLIBC_2.41 log2f64x F
+GLIBC_2.41 log2l F
+GLIBC_2.41 log2p1 F
+GLIBC_2.41 log2p1f F
+GLIBC_2.41 log2p1f128 F
+GLIBC_2.41 log2p1f32 F
+GLIBC_2.41 log2p1f32x F
+GLIBC_2.41 log2p1f64 F
+GLIBC_2.41 log2p1f64x F
+GLIBC_2.41 log2p1l F
+GLIBC_2.41 logb F
+GLIBC_2.41 logbf F
+GLIBC_2.41 logbf128 F
+GLIBC_2.41 logbf32 F
+GLIBC_2.41 logbf32x F
+GLIBC_2.41 logbf64 F
+GLIBC_2.41 logbf64x F
+GLIBC_2.41 logbl F
+GLIBC_2.41 logf F
+GLIBC_2.41 logf128 F
+GLIBC_2.41 logf32 F
+GLIBC_2.41 logf32x F
+GLIBC_2.41 logf64 F
+GLIBC_2.41 logf64x F
+GLIBC_2.41 logl F
+GLIBC_2.41 logp1 F
+GLIBC_2.41 logp1f F
+GLIBC_2.41 logp1f128 F
+GLIBC_2.41 logp1f32 F
+GLIBC_2.41 logp1f32x F
+GLIBC_2.41 logp1f64 F
+GLIBC_2.41 logp1f64x F
+GLIBC_2.41 logp1l F
+GLIBC_2.41 lrint F
+GLIBC_2.41 lrintf F
+GLIBC_2.41 lrintf128 F
+GLIBC_2.41 lrintf32 F
+GLIBC_2.41 lrintf32x F
+GLIBC_2.41 lrintf64 F
+GLIBC_2.41 lrintf64x F
+GLIBC_2.41 lrintl F
+GLIBC_2.41 lround F
+GLIBC_2.41 lroundf F
+GLIBC_2.41 lroundf128 F
+GLIBC_2.41 lroundf32 F
+GLIBC_2.41 lroundf32x F
+GLIBC_2.41 lroundf64 F
+GLIBC_2.41 lroundf64x F
+GLIBC_2.41 lroundl F
+GLIBC_2.41 modf F
+GLIBC_2.41 modff F
+GLIBC_2.41 modff128 F
+GLIBC_2.41 modff32 F
+GLIBC_2.41 modff32x F
+GLIBC_2.41 modff64 F
+GLIBC_2.41 modff64x F
+GLIBC_2.41 modfl F
+GLIBC_2.41 nan F
+GLIBC_2.41 nanf F
+GLIBC_2.41 nanf128 F
+GLIBC_2.41 nanf32 F
+GLIBC_2.41 nanf32x F
+GLIBC_2.41 nanf64 F
+GLIBC_2.41 nanf64x F
+GLIBC_2.41 nanl F
+GLIBC_2.41 nearbyint F
+GLIBC_2.41 nearbyintf F
+GLIBC_2.41 nearbyintf128 F
+GLIBC_2.41 nearbyintf32 F
+GLIBC_2.41 nearbyintf32x F
+GLIBC_2.41 nearbyintf64 F
+GLIBC_2.41 nearbyintf64x F
+GLIBC_2.41 nearbyintl F
+GLIBC_2.41 nextafter F
+GLIBC_2.41 nextafterf F
+GLIBC_2.41 nextafterf128 F
+GLIBC_2.41 nextafterf32 F
+GLIBC_2.41 nextafterf32x F
+GLIBC_2.41 nextafterf64 F
+GLIBC_2.41 nextafterf64x F
+GLIBC_2.41 nextafterl F
+GLIBC_2.41 nextdown F
+GLIBC_2.41 nextdownf F
+GLIBC_2.41 nextdownf128 F
+GLIBC_2.41 nextdownf32 F
+GLIBC_2.41 nextdownf32x F
+GLIBC_2.41 nextdownf64 F
+GLIBC_2.41 nextdownf64x F
+GLIBC_2.41 nextdownl F
+GLIBC_2.41 nexttoward F
+GLIBC_2.41 nexttowardf F
+GLIBC_2.41 nexttowardl F
+GLIBC_2.41 nextup F
+GLIBC_2.41 nextupf F
+GLIBC_2.41 nextupf128 F
+GLIBC_2.41 nextupf32 F
+GLIBC_2.41 nextupf32x F
+GLIBC_2.41 nextupf64 F
+GLIBC_2.41 nextupf64x F
+GLIBC_2.41 nextupl F
+GLIBC_2.41 pow F
+GLIBC_2.41 powf F
+GLIBC_2.41 powf128 F
+GLIBC_2.41 powf32 F
+GLIBC_2.41 powf32x F
+GLIBC_2.41 powf64 F
+GLIBC_2.41 powf64x F
+GLIBC_2.41 powl F
+GLIBC_2.41 remainder F
+GLIBC_2.41 remainderf F
+GLIBC_2.41 remainderf128 F
+GLIBC_2.41 remainderf32 F
+GLIBC_2.41 remainderf32x F
+GLIBC_2.41 remainderf64 F
+GLIBC_2.41 remainderf64x F
+GLIBC_2.41 remainderl F
+GLIBC_2.41 remquo F
+GLIBC_2.41 remquof F
+GLIBC_2.41 remquof128 F
+GLIBC_2.41 remquof32 F
+GLIBC_2.41 remquof32x F
+GLIBC_2.41 remquof64 F
+GLIBC_2.41 remquof64x F
+GLIBC_2.41 remquol F
+GLIBC_2.41 rint F
+GLIBC_2.41 rintf F
+GLIBC_2.41 rintf128 F
+GLIBC_2.41 rintf32 F
+GLIBC_2.41 rintf32x F
+GLIBC_2.41 rintf64 F
+GLIBC_2.41 rintf64x F
+GLIBC_2.41 rintl F
+GLIBC_2.41 round F
+GLIBC_2.41 roundeven F
+GLIBC_2.41 roundevenf F
+GLIBC_2.41 roundevenf128 F
+GLIBC_2.41 roundevenf32 F
+GLIBC_2.41 roundevenf32x F
+GLIBC_2.41 roundevenf64 F
+GLIBC_2.41 roundevenf64x F
+GLIBC_2.41 roundevenl F
+GLIBC_2.41 roundf F
+GLIBC_2.41 roundf128 F
+GLIBC_2.41 roundf32 F
+GLIBC_2.41 roundf32x F
+GLIBC_2.41 roundf64 F
+GLIBC_2.41 roundf64x F
+GLIBC_2.41 roundl F
+GLIBC_2.41 scalb F
+GLIBC_2.41 scalbf F
+GLIBC_2.41 scalbl F
+GLIBC_2.41 scalbln F
+GLIBC_2.41 scalblnf F
+GLIBC_2.41 scalblnf128 F
+GLIBC_2.41 scalblnf32 F
+GLIBC_2.41 scalblnf32x F
+GLIBC_2.41 scalblnf64 F
+GLIBC_2.41 scalblnf64x F
+GLIBC_2.41 scalblnl F
+GLIBC_2.41 scalbn F
+GLIBC_2.41 scalbnf F
+GLIBC_2.41 scalbnf128 F
+GLIBC_2.41 scalbnf32 F
+GLIBC_2.41 scalbnf32x F
+GLIBC_2.41 scalbnf64 F
+GLIBC_2.41 scalbnf64x F
+GLIBC_2.41 scalbnl F
+GLIBC_2.41 setpayload F
+GLIBC_2.41 setpayloadf F
+GLIBC_2.41 setpayloadf128 F
+GLIBC_2.41 setpayloadf32 F
+GLIBC_2.41 setpayloadf32x F
+GLIBC_2.41 setpayloadf64 F
+GLIBC_2.41 setpayloadf64x F
+GLIBC_2.41 setpayloadl F
+GLIBC_2.41 setpayloadsig F
+GLIBC_2.41 setpayloadsigf F
+GLIBC_2.41 setpayloadsigf128 F
+GLIBC_2.41 setpayloadsigf32 F
+GLIBC_2.41 setpayloadsigf32x F
+GLIBC_2.41 setpayloadsigf64 F
+GLIBC_2.41 setpayloadsigf64x F
+GLIBC_2.41 setpayloadsigl F
+GLIBC_2.41 signgam D 0x4
+GLIBC_2.41 significand F
+GLIBC_2.41 significandf F
+GLIBC_2.41 significandl F
+GLIBC_2.41 sin F
+GLIBC_2.41 sincos F
+GLIBC_2.41 sincosf F
+GLIBC_2.41 sincosf128 F
+GLIBC_2.41 sincosf32 F
+GLIBC_2.41 sincosf32x F
+GLIBC_2.41 sincosf64 F
+GLIBC_2.41 sincosf64x F
+GLIBC_2.41 sincosl F
+GLIBC_2.41 sinf F
+GLIBC_2.41 sinf128 F
+GLIBC_2.41 sinf32 F
+GLIBC_2.41 sinf32x F
+GLIBC_2.41 sinf64 F
+GLIBC_2.41 sinf64x F
+GLIBC_2.41 sinh F
+GLIBC_2.41 sinhf F
+GLIBC_2.41 sinhf128 F
+GLIBC_2.41 sinhf32 F
+GLIBC_2.41 sinhf32x F
+GLIBC_2.41 sinhf64 F
+GLIBC_2.41 sinhf64x F
+GLIBC_2.41 sinhl F
+GLIBC_2.41 sinl F
+GLIBC_2.41 sinpi F
+GLIBC_2.41 sinpif F
+GLIBC_2.41 sinpif128 F
+GLIBC_2.41 sinpif32 F
+GLIBC_2.41 sinpif32x F
+GLIBC_2.41 sinpif64 F
+GLIBC_2.41 sinpif64x F
+GLIBC_2.41 sinpil F
+GLIBC_2.41 sqrt F
+GLIBC_2.41 sqrtf F
+GLIBC_2.41 sqrtf128 F
+GLIBC_2.41 sqrtf32 F
+GLIBC_2.41 sqrtf32x F
+GLIBC_2.41 sqrtf64 F
+GLIBC_2.41 sqrtf64x F
+GLIBC_2.41 sqrtl F
+GLIBC_2.41 tan F
+GLIBC_2.41 tanf F
+GLIBC_2.41 tanf128 F
+GLIBC_2.41 tanf32 F
+GLIBC_2.41 tanf32x F
+GLIBC_2.41 tanf64 F
+GLIBC_2.41 tanf64x F
+GLIBC_2.41 tanh F
+GLIBC_2.41 tanhf F
+GLIBC_2.41 tanhf128 F
+GLIBC_2.41 tanhf32 F
+GLIBC_2.41 tanhf32x F
+GLIBC_2.41 tanhf64 F
+GLIBC_2.41 tanhf64x F
+GLIBC_2.41 tanhl F
+GLIBC_2.41 tanl F
+GLIBC_2.41 tanpi F
+GLIBC_2.41 tanpif F
+GLIBC_2.41 tanpif128 F
+GLIBC_2.41 tanpif32 F
+GLIBC_2.41 tanpif32x F
+GLIBC_2.41 tanpif64 F
+GLIBC_2.41 tanpif64x F
+GLIBC_2.41 tanpil F
+GLIBC_2.41 tgamma F
+GLIBC_2.41 tgammaf F
+GLIBC_2.41 tgammaf128 F
+GLIBC_2.41 tgammaf32 F
+GLIBC_2.41 tgammaf32x F
+GLIBC_2.41 tgammaf64 F
+GLIBC_2.41 tgammaf64x F
+GLIBC_2.41 tgammal F
+GLIBC_2.41 totalorder F
+GLIBC_2.41 totalorderf F
+GLIBC_2.41 totalorderf128 F
+GLIBC_2.41 totalorderf32 F
+GLIBC_2.41 totalorderf32x F
+GLIBC_2.41 totalorderf64 F
+GLIBC_2.41 totalorderf64x F
+GLIBC_2.41 totalorderl F
+GLIBC_2.41 totalordermag F
+GLIBC_2.41 totalordermagf F
+GLIBC_2.41 totalordermagf128 F
+GLIBC_2.41 totalordermagf32 F
+GLIBC_2.41 totalordermagf32x F
+GLIBC_2.41 totalordermagf64 F
+GLIBC_2.41 totalordermagf64x F
+GLIBC_2.41 totalordermagl F
+GLIBC_2.41 trunc F
+GLIBC_2.41 truncf F
+GLIBC_2.41 truncf128 F
+GLIBC_2.41 truncf32 F
+GLIBC_2.41 truncf32x F
+GLIBC_2.41 truncf64 F
+GLIBC_2.41 truncf64x F
+GLIBC_2.41 truncl F
+GLIBC_2.41 ufromfp F
+GLIBC_2.41 ufromfpf F
+GLIBC_2.41 ufromfpf128 F
+GLIBC_2.41 ufromfpf32 F
+GLIBC_2.41 ufromfpf32x F
+GLIBC_2.41 ufromfpf64 F
+GLIBC_2.41 ufromfpf64x F
+GLIBC_2.41 ufromfpl F
+GLIBC_2.41 ufromfpx F
+GLIBC_2.41 ufromfpxf F
+GLIBC_2.41 ufromfpxf128 F
+GLIBC_2.41 ufromfpxf32 F
+GLIBC_2.41 ufromfpxf32x F
+GLIBC_2.41 ufromfpxf64 F
+GLIBC_2.41 ufromfpxf64x F
+GLIBC_2.41 ufromfpxl F
+GLIBC_2.41 y0 F
+GLIBC_2.41 y0f F
+GLIBC_2.41 y0f128 F
+GLIBC_2.41 y0f32 F
+GLIBC_2.41 y0f32x F
+GLIBC_2.41 y0f64 F
+GLIBC_2.41 y0f64x F
+GLIBC_2.41 y0l F
+GLIBC_2.41 y1 F
+GLIBC_2.41 y1f F
+GLIBC_2.41 y1f128 F
+GLIBC_2.41 y1f32 F
+GLIBC_2.41 y1f32x F
+GLIBC_2.41 y1f64 F
+GLIBC_2.41 y1f64x F
+GLIBC_2.41 y1l F
+GLIBC_2.41 yn F
+GLIBC_2.41 ynf F
+GLIBC_2.41 ynf128 F
+GLIBC_2.41 ynf32 F
+GLIBC_2.41 ynf32x F
+GLIBC_2.41 ynf64 F
+GLIBC_2.41 ynf64x F
+GLIBC_2.41 ynl F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist
new file mode 100644
index 0000000000..e69de29bb2
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist
new file mode 100644
index 0000000000..e69de29bb2
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist
new file mode 100644
index 0000000000..e884cdda8b
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist
@@ -0,0 +1,55 @@
+GLIBC_2.41 __b64_ntop F
+GLIBC_2.41 __b64_pton F
+GLIBC_2.41 __dn_count_labels F
+GLIBC_2.41 __fp_nquery F
+GLIBC_2.41 __fp_query F
+GLIBC_2.41 __fp_resstat F
+GLIBC_2.41 __hostalias F
+GLIBC_2.41 __loc_aton F
+GLIBC_2.41 __loc_ntoa F
+GLIBC_2.41 __p_cdname F
+GLIBC_2.41 __p_cdnname F
+GLIBC_2.41 __p_class F
+GLIBC_2.41 __p_class_syms D 0x54
+GLIBC_2.41 __p_fqname F
+GLIBC_2.41 __p_fqnname F
+GLIBC_2.41 __p_option F
+GLIBC_2.41 __p_query F
+GLIBC_2.41 __p_rcode F
+GLIBC_2.41 __p_time F
+GLIBC_2.41 __p_type F
+GLIBC_2.41 __p_type_syms D 0x228
+GLIBC_2.41 __putlong F
+GLIBC_2.41 __putshort F
+GLIBC_2.41 __res_close F
+GLIBC_2.41 __res_hostalias F
+GLIBC_2.41 __res_isourserver F
+GLIBC_2.41 __res_nameinquery F
+GLIBC_2.41 __res_queriesmatch F
+GLIBC_2.41 __sym_ntop F
+GLIBC_2.41 __sym_ntos F
+GLIBC_2.41 __sym_ston F
+GLIBC_2.41 _getlong F
+GLIBC_2.41 _getshort F
+GLIBC_2.41 inet_net_ntop F
+GLIBC_2.41 inet_net_pton F
+GLIBC_2.41 inet_neta F
+GLIBC_2.41 ns_datetosecs F
+GLIBC_2.41 ns_format_ttl F
+GLIBC_2.41 ns_get16 F
+GLIBC_2.41 ns_get32 F
+GLIBC_2.41 ns_initparse F
+GLIBC_2.41 ns_makecanon F
+GLIBC_2.41 ns_msg_getflag F
+GLIBC_2.41 ns_name_ntol F
+GLIBC_2.41 ns_name_rollback F
+GLIBC_2.41 ns_parse_ttl F
+GLIBC_2.41 ns_parserr F
+GLIBC_2.41 ns_put16 F
+GLIBC_2.41 ns_put32 F
+GLIBC_2.41 ns_samedomain F
+GLIBC_2.41 ns_samename F
+GLIBC_2.41 ns_skiprr F
+GLIBC_2.41 ns_sprintrr F
+GLIBC_2.41 ns_sprintrrf F
+GLIBC_2.41 ns_subdomain F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist
new file mode 100644
index 0000000000..e69de29bb2
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist
new file mode 100644
index 0000000000..6d1e3b0267
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist
@@ -0,0 +1,40 @@
+GLIBC_2.41 td_init F
+GLIBC_2.41 td_log F
+GLIBC_2.41 td_symbol_list F
+GLIBC_2.41 td_ta_clear_event F
+GLIBC_2.41 td_ta_delete F
+GLIBC_2.41 td_ta_enable_stats F
+GLIBC_2.41 td_ta_event_addr F
+GLIBC_2.41 td_ta_event_getmsg F
+GLIBC_2.41 td_ta_get_nthreads F
+GLIBC_2.41 td_ta_get_ph F
+GLIBC_2.41 td_ta_get_stats F
+GLIBC_2.41 td_ta_map_id2thr F
+GLIBC_2.41 td_ta_map_lwp2thr F
+GLIBC_2.41 td_ta_new F
+GLIBC_2.41 td_ta_reset_stats F
+GLIBC_2.41 td_ta_set_event F
+GLIBC_2.41 td_ta_setconcurrency F
+GLIBC_2.41 td_ta_thr_iter F
+GLIBC_2.41 td_ta_tsd_iter F
+GLIBC_2.41 td_thr_clear_event F
+GLIBC_2.41 td_thr_dbresume F
+GLIBC_2.41 td_thr_dbsuspend F
+GLIBC_2.41 td_thr_event_enable F
+GLIBC_2.41 td_thr_event_getmsg F
+GLIBC_2.41 td_thr_get_info F
+GLIBC_2.41 td_thr_getfpregs F
+GLIBC_2.41 td_thr_getgregs F
+GLIBC_2.41 td_thr_getxregs F
+GLIBC_2.41 td_thr_getxregsize F
+GLIBC_2.41 td_thr_set_event F
+GLIBC_2.41 td_thr_setfpregs F
+GLIBC_2.41 td_thr_setgregs F
+GLIBC_2.41 td_thr_setprio F
+GLIBC_2.41 td_thr_setsigpending F
+GLIBC_2.41 td_thr_setxregs F
+GLIBC_2.41 td_thr_sigsetmask F
+GLIBC_2.41 td_thr_tls_get_addr F
+GLIBC_2.41 td_thr_tlsbase F
+GLIBC_2.41 td_thr_tsd F
+GLIBC_2.41 td_thr_validate F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist
new file mode 100644
index 0000000000..e69de29bb2
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/localplt.data b/sysdeps/unix/sysv/linux/aarch64/ilp32/localplt.data
new file mode 100644
index 0000000000..6e36c788b4
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/localplt.data
@@ -0,0 +1,10 @@
+# See scripts/check-localplt.awk for how this file is processed.
+# PLT use is required for the malloc family and for matherr because
+# users can define their own functions and have library internals call them.
+libc.so: calloc
+libc.so: free
+libc.so: malloc
+libc.so: realloc
+# If outline atomics are used, libgcc (built outside of glibc) may
+# call __getauxval using the PLT.
+libc.so: __getauxval ?
